---
title: "MICB425 Portfolio - Microbial Ecological Genomics: From Genomes to Biomes, Spring 2018"
author: "Albert Chang (26234147)"
date: "version `r format(Sys.time(), '%B %d, %Y')`"
output: 
  html_document:
    toc: true
---
# Module 01
Reserve the first level headings (#) for the start of a new Module. This will help to organize your portfolio in an intuitive fashion.  
**Note: Please edit this template to your heart's content. This is meant to be the armature upon which you build your individual portfolio. You do not need to keep this instructive text in your final portfolio, although you do need to keep module and assignment names so we can identify what is what.**


## Module 01 portfolio check
The first of your second level headers (##) is to be used for the portfolio content checks. The Module 01 portfolio check has been built for you directly into this template, but will also be available as a stand-alone markdown document available on the MICB425 GitHub so that you know what is required in each module section in your portfolio. The completion status and comments will be filled in by the instructors during portfolio checks when your current portfolios are pulled from GitHub.

* Installation check
    + Completion status:
    + Comments:
* Portfolio repo setup
    + Completion status: 
    + Comments:
* RMarkdown Pretty PDF Challenge
    + Completion status:
    + Comments:
* Evidence worksheet_01
    + Completion status:
    + Comments:
* Evidence worksheet_02
    + Completion status:
    + Comments:
* Evidence worksheet_03
    + Completion status:
    + Comments:
* Problem Set_01
    + Completion status:
    + Comments:
* Problem Set_02
    + Completion status:
    + Comments:
* Writing assessment_01
    + Completion status:
    + Comments:
* Additional Readings
    + Completion status:
    + Comments  
    
## Data science Friday  
The remaining second level headers (##) are for separating data science Friday, regular course, and project content. In this module, you will only need to include data science Friday and regular course content; projects will come later in the course.

### Installation check
Third level headers (###) should be used for links to assignments, evidence worksheets, problem sets, and readings, as seen here.  

Use this space to include your installation screenshots.  
**Sent via e-mail**



### Portfolio repo setup
Detail the code you used to create, initialize, and push your portfolio repo to GitHub. This will be helpful as you will need to repeat many of these steps to update your porfolio throughout the course.



### RMarkdown pretty PDF challenge
**Please see Assignment_3.Rmd in DS_Friday folder.**

## Origins and Earth Systems
### Evidence worksheet 01 
The template for the first Evidence Worksheet has been included here. The first thing for any assignment should link(s) to any relevant literature (which should be included as full citations in a module references section below).

You can copy-paste in the answers you recorded when working through the evidence worksheet into this portfolio template. 

As you include Evidence worksheets and Problem sets in the future, ensure that you delineate Questions/Learning Objectives/etc. by using headers that are 4th level and greater. This will still create header markings when you render (knit) the document, but will exclude these levels from the Table of Contents. That's a good thing. You don't want to clutter the Table of Contents too much.

[Whitman *et al* 1998](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC33863/)

#### Learning objectives
Describe the numerical abundance of microbial life in relation to ecology and biogeochemistry of Earth systems. 

#### General questions
* What were the main questions being asked?  
      - what is the number of prokaryotes on Earth  
      - What is the range of prokaryotes present in specific parts of the Earth  
(none of these questions were directly stated in the paper)

* What were the primary methodological approaches used?  
      - No methods were described in the paper  
      - Literature search/survey was done in order to obtain numbers for making estimates of the environments  
      - Number calculated (of hypothetical sample) was extrapolated to estimate the number of prokaryotes present  


* Summarize the main results or findings.  
      - Cellular production rate is the highest in the open ocean  
      - Prokaryotes contribute to approx. 60-100% of the total C found in plants (make up big portion of total C on Earth)  
      - Subsurface = major habitat for prokaryotes (greatest in terms of number of prokaryotes present than that found in other parts of the biosphere)  
      - Data not completely accurate since mostly estimated by extrapolation (may need to be revised)  
      - Turnover time is fastest in domestic mammals (rapidly growing population, although it's the least in terms of population size)  
      - Cellular productivity of subsurface prokaryotes comparable to that of domestic animals  


* Do new questions arise from the results?  
      - Yes. The paper estimated the number of prokaryotes. How does the number of bacteria compare to that of archaea?  
      - How does climate change affect prokaryote abundance? (Paper was published in 1998; if done again today, what would the results be and how would they compare?)  

* Were there any specific challenges or advantages in understanding the paper (*e.g.* did the authors provide sufficient background information to understand experimental logic, were methods explained adequately, were any specific assumptions made, were conclusions justified based on the evidence, were the figures or tables useful and easy to understand)?  
      - Not much background information nor description of methods were discussed in the paper  
      - Did literature survey so each method will possibly have diferences (variability of results), as well as different uncertainties (error rate). Data will vary across reported results.  
      - Didn't really state purpose of the paper, particularly in the abstract; can make purpose more explicit so that it is easier to tie it to the bigger picture and help the reader understand the importance of the study  


### Problem set 01
#### Learning objectives: 
Describe the numerical abundance of microbial life in relation to the ecology and biogeochemistry of Earth systems.

#### Specific questions:
* What are the primary prokaryotic habitats on Earth and how do they vary with respect to their capacity to support life? Provide a breakdown of total cell abundance for each primary habitat from the tables provided in the text.   


      - Aquatic: 1.181 x 10^30^ cells  
        - Freshwater - 1.31 x 10^26^ cells  
        - Saline lakes - 1.0 x 10^26^ cells  
        - Marine - 1.18 x 10^29^ cells
      - Soil: 2.56 x 10^29^ cells  
      - Subsurface: 3.8 x 10^30^ cells  
        - 10 cm marine  
        - 8 m terrestrial  


* What is the estimated prokaryotic cell abundance in the upper 200 m of the ocean and what fraction of this biomass is represented by marine cyanobacterium including Prochlorococcus? What is the significance of this ratio with respect to carbon cycling in the ocean and the atmospheric composition of the Earth?  

      - cell density: 5 x 10^5^ cells/mL  
      - Prochlorococcus spp. -> cell density: 4 x 10^4^ cells/mL  
      - fraction = (4 x 10^4^ cells/mL) / (5 x 10^5^ cells/mL) = 0.08 = 8 % marine cyanobacteria
      - Significant portion of upper oceanic microbes are photosynthetic cyanobacteria needed to fix carbon -> provides C for everyone = high turnover rate  
      

* What is the difference between an autotroph, heterotroph, and a lithotroph based on information provided in the text?  

    - Autotroph: "Self-nourishing"  
                  - Fix inorganic carbon (CO~2~)  
    - Heterotroph: assimilate organic carbon  
    - Lithotroph: use inorganic substrates (carbon) for energy source    

* Based on information provided in the text and your knowledge of geography what is the deepest habitat capable of supporting prokaryotic life? What is the primary limiting factor at this depth?  

      - Deep habitats supporting life:  
      - Subsurface: Terrestrial = 4 km from sediment (a bit below Mariana's Trench)  
      - Temperature is limiting factor (approx. 125 degrees Celsius); change in temp. is approx. 22 degrees Celsius per km depth.  


* Based on information provided in the text your knowledge of geography what is the highest habitat capable of supporting prokaryotic life? What is the primary limiting factor at this height?  

      - 57-77 km above Earth's surface (a bit above Mt. Everest)  
      - Limiting factor is radiation  

* Based on estimates of prokaryotic habitat limitation, what is the vertical distance of the Earth's biosphere measured in km?  

      - 61-81 km = range of Earth's biosphere

* How was annual cellular production of prokaryotes described in Table 7 column four determined? (Provide an example of the calculation)  

      - (Population) x (Turnover/yr) = Cells/yr  
      - Marine Heterotrophs:  
        - (3.6 x 10^28^ cells x 365 days) / 16 turnovers = 8.2 x 10^29^  


* What is the relationship between carbon content, carbon assimilation efficiency and turnover rates in the upper 200m of the ocean? Why does this vary with depth in the ocean and between terrestrial and marine habitats?  

      - C content = amount of organic C stored
      - C assimilation efficiency = inorganic C -> organic C
      - Turnover rate = fraction of C that leaves a reservoir per specified time
      - Carbon efficiency = 20%  
        - 5-20 fg C per Cell = 20 x 10^-30^ Pg/cell  
        - (3.6 x 10^28^ cells) x (20 x 10^-30^ Pg/cell) = 0.72 Pg C in marine
        - Heterotrophs:  
          - 4 x 0.72 = 2.88 Pg/yr  
      - (C consumed) / (turnover rate)
      - more sunlight near surface
      - high nutrient = high energy = high turnover rate  

* How were the frequency numbers for four simultaneous mutations in shared genes determined for marine heterotrophs and marine autotrophs given an average mutation rate of 4 x 10^-7^ per DNA replication? (Provide an example of the calculation with units. Hint: cell and generation cancel out)
     - [( (4 x 10^-7^ mutation rate per gene) /cell)^4^ x ( (3.6 x 10^28^ cells) /16 days) x (1 day/24 hrs)]^-1^ = 0.42 hrs for 4 mutations    
     
     
* Given the large population size and high mutation rate of prokaryotic cells, what are the implications with respect to genetic diversity and adaptive potential? Are point mutations the only way in which microbial genomes diversify and adapt?
     - Point mutations are not the only way. Horizontal and lateral gene transfer can also occur. Conjugation by plasmid, transduction by virus, and transformation by naked virus can all occur. High genetic diversity and adaptive potential may possibly lead to new species.    
     
     
* What relationships can be inferred between prokaryotic abundance, diversity, and metabolic potential based on the information provided in the text? 
    - Upper 200 m of marine habitat has higher genetic diversity than in domestic animals. 
    - Open ocean prokaryotes have much greater adaptive potential than those in soil, subsurface and domestic animals. 
    - Greater prokaryotic abundance in unconsolidated subsurface than in soil.


### Evidence Worksheet_02 "Life and the Evolution of Earth's Atmosphere"
#### Learning objectives: 
Comment on the emergence of microbial life and the evolution of Earth systems

![](1.jpg){height=1200px, width=450px}
![](2.jpg){height=1200px, width=450px}
![](3.jpg){height=1200px, width=450px}
![](4.jpg){height=1600px, width=450px}
    
### Problem set_02 "Microbial Engines"
#### Learning objectives: 
Discuss the role of microbial diversity and formation of coupled metabolism in driving global biogeochemical cycles.

#### Specific Questions:
* What are the primary geophysical and biogeochemical processes that create and sustain conditions for life on Earth? How do abiotic versus biotic processes vary with respect to matter and energy transformation and how are they interconnected?  

    - geochemical cycles:  
        - tectonics & atmospheric photochemical processes  
            - supply substrates and remove products
        - allow elements and molecules to interact and form/break bonds in a cyclical manner  
        - Most H~2~ in mantle escape -> most geochemical reactions based on acid/base reactions (transfer of H^+^ without e^-^)  
        - volcanism and rock weathering resupply C, S, P
      
    - biochemical cycles: redox reactions
        - transfer e^-^ and H^+^ from limited set of elements
        - microbially catalyzed, thermodynamically constrained redox reactions (coupled half-cells)

* Why is Earth's redox state considered an emergent property?

    - abiotic (acid/base) and biotic (redox) reactions altered surface redox state of the planet. Microbes took advantage of Earth's acid/base redox energy.  
      --> The evolution of the processes created the redox condition of oceans and atmosphere
      
      - E.g. thermodynamic equilibrium would lead to eventual depeletion of substrates essential for life -> biological oxidation of Earth driven by photosynthesis by microbes (= energy transduction process not directly dependent on preformed bond energy)
  

* How do reversible electron transfer reactions give rise to element and nutrient cycles at different ecological scales? What strategies do microbes use to overcome thermodynamic barriers to reversible electron flow?  
    - forward and reverse reactions required to maintain cycle
        - E.g. Methanogenic Archaea: CO~2~ + H~2~ -> CH~4~
            - If H~2~ is low, then the reverse reaction becomes thermodynamically favourable and is done by another Archaea using the methanogic machinery in reverse.
    - sunlight replenishes energy that is lost
    - cooperation between deifferent species to overcome thermodynamic barriers

* Using information provided in the text, describe how the nitrogen cycle partitions between different redox 'niches' and microbial groups. Is there a relationship between the nitrogen cycle and climate change?   

    - nitrogen fixation = the only biological process that makes N~2~ accessible for sysnthesis of proteins/nucleic acids (N~2~ -> NH~4~^+^) -> biologically irreversible reaction catalyzed by nitrogenase (conserved enzyme), which is inhibited by O~2~  
    - In presence of O~2~, NH~4~^+^ is oxidzied to NO~2~^-^ to NO~3~^-^, all using a specific group of Bacteria or Archaea at each step. This also reduces CO~2~ to organic C at the same time.  
    - In the absence of O~2~, another set of microbes use NO~2~^-^ and NO~3~^-^ as electron acceptors in anaerobic oxidation of organic C and also forms N~2~ (repeating the N cycle)
    - The N cycle is spatially separated, but forms interdependent e^-^ pool influenced by photosynthetic production of O^2^ and available organic matter
    - N~2~O is a greenhouse gas and can contribute to global warming. Also, micrboes in the N cycle convert organic C to CO~2~, which may impact the C cycle and climate change.  
    

* What is the relationship between microbial diversity and metabolic diversity and how does this relate to the discovery of new protein families from microbial community genomes?  

    - essential pathways: are conserved
    - but as microbial diversity increases, the number of new proteins discovered increases (new number of proteins increase linearly with number of new genomes sequenced)

* On what basis do the authors consider microbes the guardians of metabolism?  
    - vertical and horizontal gene transfer allows the transfer and spread of genes needed for metabolic pathways. As such, the extinction of individual taxonomic units would not result in the extinction of core machines of the pathway.
    - machines, unlike microbes, can't cycle waste back as substrate  

###Evidence worksheet_03 "The Anthropocene"  

####Learning objectives:   

Evaluate human impacts on the ecology and biogeochemistry of Earth systems.  

####General Questions:  


* What were the main questions being asked?  

    - Have humans changed the Earth system so much that the geological deposits are distinct from the Holocene & earlier epochs?  
    - If so, when did this stratigraphic signal become recognizable worldwide?  

* What were the primary methodological approaches used?  
    - There was no method, but they reviewed data from previously published papers  
    - They analyzed the published data to summarize the effects humans have on the environment and also to draw a timeline of events  
    - Used stratigraphic record to compare to previous epochs  

* Summarize the main results or findings.  
    - Anthropocene: period marked by humans that changed Earth's history (occurred recently)
    - invent of widespread and persistent materials; rapid production of those material leaves identifiable fossil and geochemical records (e.g. aluminum, plastics)  
    - modification of sedimentary processes  
        - transformed > 50% of Earth's land for: landfills, urban structures, mining, cultivation, etc.  
        - rising sea levels, eutrophication, coral bleaching, etc.  
    - N and P in soils doubled due to increased fertilizer use  
    - reactive N amount increased by 120% relative to Holocene baseline  
    - Sea-level rise  
    - Changes in climate and environment exceed Late Holocene changes  
    - Radiogenic signatures and radionuclides in sediments/ice  
        - excess ^14^C (suggested as potential marker for start of Anthropocene)  
        - ^239+240^Pu fallout from nuclear weapons testing  
    - CO~2~ above 400ppm  

* Do new questions arise from the results?  
    - What are current innovations/strategies employed to slow down/solve the problems?  
    - Although the authors listed various changes caused by humans, how would the authors officially define the start of Anthropocene? What is the boundary?  
    - What other purposes are there to label this era as Anthropocene other than for historic purposes?  

* Were there any specific challenges or advantages in understanding the paper (e.g. did the authors provide sufficient background information to understand experimental logic, were methods explained adequately, were any specific assumptions made, were conclusions justified based on the evidence, were the figures or tables useful and easy to understand)?  

    - the author provided sufficient background information and data to reinforce their points  
    - There were many figures, but they only referred to them briefly or not much at all during the discussion. So it felt like the figures were stand alones (although they are related to the text).  
    - The figures had good descriptions and were easy to understand.  
    

### Module_1 essay

Human beings are innovative creatures that have engineered many machines for technological advancement and manipulation of the environment, be it for better or for worse. That being said, are humans actually at the helm of Spaceship Earth, or are we only capable of living in the presence of a microbial world? We have utilized microbes to our advantage in innumerable processes, ranging from legume cropping in agriculture to fermentation in alcohol production, as well as denitrification in sewage treatment. As a matter of fact, humans have been relying on microbes prior to our acknowledgement of their existence. Microbes have been present on Mother Earth long before the existence of Homo sapiens and have established the distinctive, phenotypic features of Earth's atmosphere suitable for human life. In addition, microbes are an essential part of Earth's biogeochemical cycles, which are required to sustain the lives of the many other organisms which benefit the human species. Thus, humans live in a microbial world and would not be able to survive otherwise without the metabolic processes provided by microbes. Microbes, on the other hand, would easily thrive in a human-less world.  

Microbes have existed long before the birth of the human race and can easily continue to thrive even in the absence of human intervention. To begin with, it is important to discuss the history of microbial life in order to understand and appreciate the influence these microorganisms have on Earth and her environment. According to geological evidence, microbial activity has been present on Earth for at least 3.5 billion years (Gyr) (5). Evidence derived from stable-isotope fractionation of 3.5-Gyr stromatolites suggested possible sulfate reduction methanogenesis activity (2). In addition, molecular fossils from late-Archaean rocks (i.e. 3.0 - 2.5 Gyr ago) provided evidence of preserved biomarkers, such as 2??-methylhopanes from cyanobacteria, allowing for further inference of the types of microbes and metabolic processes present during that period (5). Furthermore, microbes can influence their surrounding environment through the production of metabolic products and byproducts required for survival. An example would be the nitrogen cycle, where multiple species of microbes interact together, forming a nutrient cycle. These interdependent relationships where a species of microbes utilize the waste of another species as substrate to generate energy for growth and reproduction have existed long before human intervention, and would continue to exist due the self-sustainability of the symbiosis that exists between these microbial colonies (2). Despite many volcanic eruptions, meteorite impacts and extinction events during Earth's early years, many diverse microbes persisted and continue to flourish today. In addition to being diverse and possessing remarkable adaptability, microbes have the ability to perform horizontal gene transfer (HGT) of genes to other organisms in order to preserve essential metabolic pathways (2). If the extinction, per se, of the original taxonomic unit were to occur, the dispersion of essential core genes by HGT ensures the conservation of core metabolic pathways due to the selective pressure of the environment. In other words, non-essential and irrelevant genes are lost, while those that are important are preserved. Microbes have a long history on Earth and have played an important role in shaping the planet which we currently inhabit.  

The human race as a whole is dependent on microbial activity for survival on Earth. First, microbes have established the current phenotypic atmosphere of Earth, allowing human existence. The atmosphere of the early Earth was anoxic and had a completely different composition than the atmosphere we have today (4). Abiotic and biotic factors such as volcanic eruptions and methane-producing microbes contributed to an atmosphere rich in carbon dioxide and methane (2). However, 2.3 Gyr ago, there was an initial rise in oxygen largely due to oxygenic photosynthesis performed by cyanobacteria (4). Although the rise in oxygen concentration led to a mass extinction known as the Great Oxygenation Event, the oxic atmosphere allowed the evolution of many aerobic organisms and brought about the species which would eventually be known as Homo sapiens (3).  

In addition to establishing an atmosphere suitable for humans, microbes were capable of taking advantage of the abiotic geochemical reactions produced by Earth and have co-evolved to form biogeochemical cycles (2). These biogeochemical cycles are essential for the production of major elements required for building biological macromolecules such as proteins, lipids and nucleic acids and thus, are crucial for sustaining human life (2). Biogeochemical cycles are largely dependent on reduction-oxidation (redox) reactions mediated by microbes. These important cycles create fluxes of major essential elements such as hydrogen (H), carbon (C), nitrogen (N), oxygen (O) and sulfur (S) that are required for the synthesis of all biological macromolecules and ultimately, human survival (2). An example of a major biogeochemical cycle is the nitrogen cycle. The N cycle is formed by microbes and is essential for providing living organisms access to nitrogen, needed for the synthesis nucleic acid and proteins (1). Although approximately 78% of Earth's atmosphere is composed of molecular nitrogen (N2), atmospheric nitrogen is inert and not readily available for human use (5). Nitrogen fixation is the only biological process currently available which allows the conversion of N2 to ammonium (NH4+), providing accessible nitrogen to humans, animals and non-nitrogen fixing plants altogether (2). Certain microbes, such as cyanobacteria, express nitrogenases and can catalyze the irreversible reaction to fix nitrogen (5). Following nitrogen fixation, nitrification and denitrification by certain microbes occur to convert the available nitrogen back into N2 to maintain the metabolic cycle and a balance between atmospheric nitrogen and available nitrogen (2). With the rapid growth of the human population, food becomes scarce and methods for increasing provisions are needed. Taking advantage of the endosymbiotic nitrogen-fixing microbes found in legumes, humans have developed sustainable approaches for agriculture, such as legume cropping in lieu of synthetic fertilizers, in order to provide accessible nitrogen to crops (1). Thus, microbes have played and continue to play an integral role in the storybook of human survival.  

In conclusion, microbes have played a vital part in providing the appropriate environment required for human existence and continue to play an integral part for human life on Earth. Not only have microbes lived long before the appearance of Homo sapiens, microbes have evolved and adapted to the harsh geological environment-be it by natural causes or by human influence-and have changed the phenotype of Earth's atmosphere to establish a habitable place for humans today. Lastly, microbial metabolic pathways are necessary to maintain biogeochemical cycles. This allows microbes to catalyze thermodynamically constrained redox reactions of the cycles, allowing their own survival and that of other living organisms.  


### Module 01 references
Utilize this space to include a bibliography of any literature you want associated with this module. We recommend keeping this as the final header under each module.

  - Whitman WB, Coleman DC, and Wiebe WJ. 1998. Prokaryotes: The unseen majority. Proc Natl Acad Sci USA. 95(12):6578â€“6583.  [PMC33863](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC33863/)   
  - Achenbach, J. (2012). Spaceship Earth: A new view of environmentalism. Washington Post (January 2, 2012).  
  - Falkowski, P. G., Fenchel, T., & Delong, E. F. (2008). The microbial engines that drive Earth's biogeochemical cycles. science, 320(5879), 1034-1039. [link](http://science.sciencemag.org/content/320/5879/1034)  
  - Canfield, D. E., Glazer, A. N., & Falkowski, P. G. (2010). The evolution and future of Earth's nitrogen cycle. science, 330(6001), 192-196. [link](http://science.sciencemag.org/content/330/6001/192)  
  - Kasting, J. F., & Siefert, J. L. (2002). Life and the evolution of Earth's atmosphere. Science, 296(5570), 1066-1068. [link](http://science.sciencemag.org/content/296/5570/1066)  
  - Nisbet, E. G., & Sleep, N. H. (2001). The habitat and nature of early life. Nature, 409(6823), 1083. [link](https://www.nature.com/articles/35059210)  
  - Leopold, A. (2014). The land ethic. In The Ecological Design and Planning Reader (pp. 108-121). Island Press, Washington, DC. [link](https://link.springer.com/chapter/10.5822/978-1-61091-491-8_12)  
  - Falkowski, P., Scholes, R. J., Boyle, E. E. A., Canadell, J., Canfield, D., Elser, J., ... & Mackenzie, F. T. (2000). The global carbon cycle: a test of our knowledge of earth as a system. science, 290(5490), 291-296. [link](http://science.sciencemag.org/content/290/5490/291)  
  - Kallmeyer, J., Pockalny, R., Adhikari, R. R., Smith, D. C., & D'Hondt, S. (2012). Global distribution of microbial abundance and biomass in subseafloor sediment. Proceedings of the National Academy of Sciences, 109(40), 16213-16216. [link](http://www.pnas.org/content/109/40/16213.short)  
  - Rockström, J., Steffen, W., Noone, K., Persson, Å., Chapin III, F. S., Lambin, E. F., ... & Nykvist, B. (2009). A safe operating space for humanity. nature, 461(7263), 472. [link](https://www.nature.com/articles/461472a)   
  - Schrag, D. P. (2012). Geobiology of the Anthropocene. Fundamentals of geobiology, 425-436.[link](https://onlinelibrary.wiley.com/doi/10.1002/9781118280874.ch22)   
  - Waters, C. N., Zalasiewicz, J., Summerhayes, C., Barnosky, A. D., Poirier, C., Galuszka, A., ... & Jeandel, C. (2016). The Anthropocene is functionally and stratigraphically distinct from the Holocene. Science, 351(6269), aad2622. [link](http://science.sciencemag.org/content/351/6269/aad2622)  
  - Zehnder, A. J. (1988). Biology of anaerobic microorganisms. John Wiley and Sons Inc.. [link](https://www.cabdirect.org/cabdirect/abstract/19911956781)   

References for Essay:  
1. Canfield, DE, Glazer, AN, Falkowski, PG. 2010. The Evolution and Future of Earth's Nitrogen Cycle. Science. 330:192-196. doi: 10.1126/science.1186120.  
2. Falkowski, PG, Fenchel, T, Delong, EF. 2008. The Microbial Engines That Drive Earth's Biogeochemical Cycles. Science. 320:1034-1039. doi: 10.1126/science.1153213.  
3. Fischer, WW, Hemp, J, Valentine, JS. 2016. How did life survive Earth's great oxygenation? Curr. Opin. Chem. Biol. 31:166-178. doi: 10.1016/j.cbpa.2016.03.013.  
4. Kasting, J, Siefert, J. 2003. Life and the evolution of Earth's atmosphere (vol 296, pg 1066, 2002). Science. 299:1015-1015.  
5. Sleep, NH, Nisbet, EG. 2001. The habitat and nature of early life. Nature. 409:1083-1091. doi: 10.1038/35059210.  


# Module 02


## Remapping the Body of the World

### Problem set_03 "Metagenomics: Genomic Analysis of Microbial Communities"
#### Learning objectives: 
Specific emphasis should be placed on the process used to find the answer. Be as comprehensive as possible e.g. provide URLs for web sources, literature citations, etc.  
*(Reminders for how to format links, etc in RMarkdown are in the RMarkdown Cheat Sheets)*

#### Specific Questions:
* How many prokaryotic divisions have been described and how many have no cultured representatives (microbial dark matter)?  
    - Solden et al. (2016). "The Bright Side of Microbial Dark Matter: Lessons Learned from the Uncultivated Majority". -> 89 Bacterial and 20 Archaeal phyla recogized by small rRNA subunits
        - only 0.1 - 1% of microbes in environment have been cultured
    - Youssef et al. (2015). "Assessing the Global Phylum Level Diversity Within the Bacterial Domain: A Review"
        - 70 bacterial phyla with no cultured representatives (this number may be greater)
        - previously confined to pure cultures, found 12 bacterial phyla with cultured representatives: Actinobacteria, Bacteroidetes, Chlamydiae, Chlorobi, Chloroflexi, Cyanobacteria, Thermi (Deinococcus-Thermus), Firmicutes, Planctomycetes, Proteobacteria, Spirochaetes, Thermotogae
  

* How many metagenome sequencing projects are currently available in the public domain and what types of environments are they sourced from?  
    - From EBI Metagenomics (Europe)
        - 1486 public projects
        - 86201 samples
        - soil, marine, grassland, fecal, rumens, agriculture, etc.
    - JGI (U.S.)
        - 36715 public projects

* What types of on-line resources are available for warehousing and/or analyzing environmental sequence information (provide names, URLS and applications)?  
    - MEGAN (metagenome analyzer)
        - http://ab.inf.uni-tuebingen.de/software/megan5/download/welcome.html
        - analyze metagenomes and group using taxonomic analysis
    - IMG/M (Integrated Microbial Genomes)
        - http://img.jgi.doe.gov/cgi-bin/m/main.cgi
        - tool for analyzing publicly available genome/metagenome datasets
    - MG-RAST (metagenomics analysis server)
        - https://www.mg-rast.org/
    - Tag-based sequencing: SSU rRNA -> MOTHUR, QIIME
    - BLAST (NCBI)

* What is the difference between phylogenetic and functional gene anchors and how can they be used in metagenome analysis?  
    - Phylogenetic anchors:
        - link other unknown genes within genome to a taxon
        - collection of genes can be placed into bins of discrete taxonomic groups
        - only one copy per cell; ensures no HGT, and genes only appear once in tree
        - Looks at size of bins for relative abundance of taxa
    - Functional anchors:
        - tells what the cell does
        - search for gene that codes for protin involved in the terminal steps of a pathway
        - can count abundance of phylogenetic anchor

* What is metagenomic sequence binning? What types of algorithmic approaches are used to produce sequence bins? What are some risks and opportunities associated with using sequence bins for metabolic reconstruction of uncultivated microorganisms?  
    - binning: grouping reads or contigs and assigning them to operational taxonomic units (OTU)
        - Taxonomy dependent - classifies DNA fragments by comparing to reference database
        - Taxonomy independent - reference free binning by clustering similar features (e.g. GC content, etc.)
    - sequence composition based binning -> genomic signatures
        - Interpolated Markov Model (IMM)
    - abundance based binning
        - Expectation-Maximization (EM) algorithm
    -Hybrid binning
        - Lander-Waterman model
    Sedlar et al. (2016) "Bioinformatics strategies for taxonomy independent binning and visualization of sequence in shotgun metagenomics"

* Is there an alternative to metagenomic shotgun sequencing that can be used to access the metabolic potential of uncultivated microorganisms? What are some risks and opportunities associated with this alternative?  
    - Douterelo et al. (2014). "Methodological approaches for studying the microbial ecology of drinking water distribution systems."
    - Clone dependent shotgun library
        - Find functions of proteins from the gene of uncultivable microbes by cloning gene fragments into an expression host
        - Limitation: gene may not be compatible with the host
    - Pacific Bioscience
    - Nanopore
    - Metatranscriptomes
        - RT-qPCR:
            -high sensitivity, can run multiple samples + analyze different genes at once
            - quantitative representation of changes in gene expression due to different treaments/controls
            - Limitations:
                - RNA is highly unstable
                - DNA extracted needs to be pure
        - Functional Microarrays:
            - Overall assessment of gene expression within a microbial community
            - simultaneous evaluation of many mRNA (rapid)
            - Limitations:
                - liquid chromatography/2D gels for protein separation can be tedious
                - Must use LC-MS (Liquid chromatography-mass spectrometry) to characterize samples

### Evidence worksheet_04 "Bacterial Rhodopsin Gene Expression"
#### Learning objectives: 

* Discuss the relationship between microbial community structure and metabolic diversity

* Evaluate common methods for studying the diversity of microbial communities

*	Recognize basic design elements in metagenomic workflows

#### General Questions:

* What were the main questions being asked?  
    - To describe and understand more about proteorhodopsins (PRs), photosystem genetics and biochemistry (foud that proteorhodopsin isn't needed, but provides extra energy in form of ATP)  
    - What is the physiological basis of light-activated growth stimulation and the function of the various PR photosystems?
    - Can we recover a functional PR-system in a metagenomic screening context?
    - Can a single genetic event (HGT/LGT operon) confer PR-system functionality?
    
* What were the primary methodological approaches used?  
    - Created fosmid library (gene library) -> clone in E. coli -> Screen by colour change -> extract carotenoid gene
    - Sequenced to compreltion using Tn5-Seq
        - Use copy control and induced with arabinose
            - slight colour shift may have been hard to discern (so 3/12000 hits)
    - Proton-Pumping experiment and ATP measurements
    - HPLC Analysis of metabolites

* Summarize the main results or findings.  
    - Only 6 genes are required to enable light-activated proton translocation and photophosphorylation
        - these 6 genes found in marine bacterial taxa
            - these are necessary and sufficient for complete synthesis and assembly of a fully functional PR photoprotein in E. coli
    - demonstrated that illumination of cells expressing a native marine bacterial PR photosystem generates a PMF that drives celluar ATP synthesis
    - a single genetic event can result in the acquisition of phototrophic capabilities in chemoorganotrophic microorganisms

* Do new questions arise from the results?  
    - Paper mentioned that light stimulation/yield in Pelagibacter ubique in previous study had a negative result that may be due to the influence of various factors in seawater. Are there perhaps other reasons it didn't work on P. ubique?
    - Why were only 3/12280 clone identified?

* Were there any specific challenges or advantages in understanding the paper (e.g. did the authors provide sufficient background information to understand experimental logic, were methods explained adequately, were any specific assumptions made, were conclusions justified based on the evidence, were the figures or tables useful and easy to understand)?  
    - Sufficient background information was provided
    - The figures and figure font were a bit too small to read without having to zoom in 300%
    
    
### Module 02 references
Utilize this space to include a bibliography of any literature you want associated with this module. We recommend keeping this as the final header under each module.

  - Madsen, E. L. (2005). Identifying microorganisms responsible for ecologically significant biogeochemical processes. Nature Reviews Microbiology, 3(5), 439. [link](https://www.nature.com/articles/nrmicro1151)  
  - Taupp, M., Mewis, K., & Hallam, S. J. (2011). The art and design of functional metagenomic screens. Current opinion in biotechnology, 22(3), 465-472. [link](https://www.sciencedirect.com/science/article/pii/S0958166911000371)  
  - Martinez, A., Bradley, A. S., Waldbauer, J. R., Summons, R. E., & DeLong, E. F. (2007). Proteorhodopsin photosystem gene expression enables photophosphorylation in a heterologous host. Proceedings of the National Academy of Sciences, 104(13), 5590-5595. [link](http://www.pnas.org/content/104/13/5590.short)  
  - Wooley, J. C., Godzik, A., & Friedberg, I. (2010). A primer on metagenomics. PLoS computational biology, 6(2), e1000667. [link](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1000667)  


---

# Module 03


## Microbial Species Concepts

---

### Problem set_04 "Fine-scale phylogenetic architecture""
#### Learning objectives: 

* Gain experience estimating diversity within a hypothetical microbial community

#### Part 1: Description and enumeration
Obtain a collection of "microbial" cells from "seawater". The cells were concentrated from different depth intervals by a marine microbiologist travelling along the Line-P transect in the northeast subarctic Pacific Ocean off the coast of Vancouver Island British Columbia. 

Sort out and identify different microbial "species" based on shared properties or traits. Record your data in this Rmarkdown using the example data as a guide.

Once you have defined your binning criteria, separate the cells using the sampling bags provided. These operational taxonomic units (OTUs) will be considered separate "species". This problem set is based on content available at [What is Biodiversity](http://cnx.org/content/m12179/latest/).

For example, load in the packages you will use.
```{r message=FALSE}
#To make tables
library(kableExtra)
library(knitr)
#To manipulate and plot data
library(tidyverse)
```

Then load in the data. You should use a similar format to record your community data.
```{r}
example_data1 = data.frame(
  number = c(1,2,3),
  name = c("lion", "tiger", "bear"),
  characteristics = c("brown cat", "striped cat", "not a cat"),
  occurences = c(2, 4, 1)
)
```

Finally, use these data to create a table.
```{r}
example_data1 %>% 
  kable("html") %>%
  kable_styling(bootstrap_options = "striped", font_size = 10, full_width = F)
```
	
For your community:

* Construct a table listing each species, its distinguishing characteristics, the name you have given it, and the number of occurrences of the species in the collection.
* Ask yourself if your collection of microbial cells from seawater represents the actual diversity of microorganisms inhabiting waters along the Line-P transect. Were the majority of different species sampled or were many missed? 

```{r}
candy_counts = read.table(file="Candy_community.csv", header=TRUE, sep=",", na.strings=c("NAN", "NA", "."))
candy_table = candy_counts %>% kable("html")

#The collection of microbial cells from seawater does not represent the actual diversity of microorganisms inhabtigin waters along the Line-P transect. Many were missed, especially if the abundance was low for those species in the first place.
```
 
#### Part 2: Collector's curve
To help answer the questions raised in Part 1, you will conduct a simple but informative analysis that is a standard practice in biodiversity surveys. This analysis involves constructing a collector's curve that plots the cumulative number of species observed along the y-axis and the cumulative number of individuals classified along the x-axis. This curve is an increasing function with a slope that will decrease as more individuals are classified and as fewer species remain to be identified. If sampling stops while the curve is still rapidly increasing then this indicates that sampling is incomplete and many species remain undetected. Alternatively, if the slope of the curve reaches zero (flattens out), sampling is likely more than adequate.

To construct the curve for your samples, choose a cell within the collection at random. This will be your first data point, such that X = 1 and Y = 1. Next, move consistently in any direction to a new cell and record whether it is different from the first. In this step X = 2, but Y may remain 1 or change to 2 if the individual represents a new species. Repeat this process until you have proceeded through all cells in your collection.

For example, we load in these data.
```{r}
example_data2 = data.frame(
  x = c(1,2,3,4,5,6,7,8,9,10),
  y = c(1,2,3,4,4,5,5,5,6,6)
)
```

And then create a plot. We will use a scatterplot (geom_point) to plot the raw data and then add a smoother to see the overall trend of the data.
```{r}
ggplot(example_data2, aes(x=x, y=y)) +
  geom_point() +
  geom_smooth() +
  labs(x="Cumulative number of individuals classified", y="Cumulative number of species observed")
```

For your sample:

* Create a collector's curve for your sample (not the entire original community).
* Does the curve flatten out? If so, after how many individual cells have been collected?
* What can you conclude from the shape of your collector's curve as to your depth of sampling?
```{r}
candy_count_collector = data.frame(
  x = c(1,13,14,15,27,37,49,50,52,55,58,67,76,93,104,113,122,127,128,129,137,138,139,141,150,154,160,164,165,167,171,177,182,184),
  y = c(1,2,3,4,4,5,6,6,6,7,7,7,8,8,8,8,8,8,8,9,9,9,10,10,10,10,10,10,11,11,11,11,11,11)
)

ggplot(candy_count_collector, aes(x=x, y=y)) +
  geom_point() +
  geom_smooth() +
  labs(x="Cumulative number of individuals classified", y="Cumulative number of species observed")

# The curve flattens out at around 10 species, when the number of cells collected is 85.
# In the beginning and towards the end, sampling individuals increase the species moreso than in the middle. This is probably because at the start there is more chance to get cells that aren't of the same species. Likewise, near the end when there are only a few cells remaining, those cells from species that have few individuals will eventually be sampled. In the middle, there is a greater likelihood of sampling cells from the same species, which explains the plateaur near the centre of the curve.
```
#### Part 3: Diversity estimates (alpha diversity)
Using the table from Part 1, calculate species diversity using the following indices or metrics.

##### Diversity: Simpson Reciprocal Index

$\frac{1}{D}$ where $D = \sum p_i^2$

$p_i$ = the fractional abundance of the $i^{th}$ species

For example, using the example data 1 with 3 species with 2, 4, and 1 individuals each, D =

```{r}
species1 = 2/(2+4+1)
species2 = 4/(2+4+1)
species3 = 1/(2+4+1)

1 / (species1^2 + species2^2 + species3^2)
```

```{r}
#Sample

Gummy_bear = 23/184
Gummy_sour_bears = 1/184
Gummy_Rods = 48/184
Large_gummy = 0/184
Gummy_sour_swirls = 1/184
Gummy_spiders = 0/184
Gummy_cokes= 0/184
Gummy_string = 0/184
Small_brick = 4/184
Large_brick = 0/184
Skittles = 49/184
MM = 49/184
Twizzlers = 0/184
Kisses = 1/184
Gummy_balls = 6/184
Gummy_fruit = 1/184
Mutated = 1/184

Simpsons = 1/ (Gummy_bear^2 + Gummy_sour_bears^2 + Gummy_Rods^2 + Large_gummy^2 + Gummy_sour_swirls^2 + Gummy_spiders^2 + Gummy_cokes^2 + Gummy_string^2 + Small_brick^2 + Large_brick^2 + Skittles^2 + MM^2 + Twizzlers^2 + Kisses^2 + Gummy_balls^2 + Gummy_fruit^2 + Mutated^2)
```

```{r}
#Community

C_Gummy_bear = 105/793
C_Gummy_sour_bears = 3/793
C_Gummy_Rods = 174/793
C_Large_gummy = 2/793
C_Gummy_sour_swirls = 3/793
C_Gummy_spiders = 6/793
C_Gummy_cokes= 3/793
C_Gummy_string = 7/793
C_Small_brick = 15/793
C_Large_brick = 3/793
C_Skittles = 192/793
C_MM = 222/793
C_Twizzlers = 14/793
C_Kisses = 16/793
C_Gummy_balls = 24/793
C_Gummy_fruit = 2/793
C_Mutated = 2/793

C_Simpsons = 1/ (C_Gummy_bear^2 + C_Gummy_sour_bears^2 + C_Gummy_Rods^2 + C_Large_gummy^2 + C_Gummy_sour_swirls^2 + C_Gummy_spiders^2 + C_Gummy_cokes^2 + C_Gummy_string^2 + C_Small_brick^2 + C_Large_brick^2 + C_Skittles^2 + C_MM^2 + C_Twizzlers^2 + C_Kisses^2 + C_Gummy_balls^2 + C_Gummy_fruit^2 + C_Mutated^2)
```

The higher the value is, the greater the diversity. The maximum value is the number of species in the sample, which occurs when all species contain an equal number of individuals. Because the index reflects the number of species present (richness) and the relative proportions of each species with a community (evenness), this metric is a diveristy metric. Consider that a community can have the same number of species (equal richness) but manifest a skewed distribution in the proportion of each species (unequal evenness), which would result in different diveristy values.

* What is the Simpson Reciprocal Index for your sample?  
    - 4.401  

* What is the Simpson Reciprocal Index for your original total community?  
    - 4.881  

##### Richness: Chao1 richness estimator
Another way to calculate diversity is to estimate the number of species that are present in a sample based on the empirical data to give an upper boundary of the richness of a sample. Here, we use the Chao1 richness estimator. 

$S_{chao1} = S_{obs} + \frac{a^2}{2b})$

$S_{obs}$ = total number of species observed
a = species observed once
b = species observed twice or more

So for our previous example community of 3 species with 2, 4, and 1 individuals each, $S_{chao1}$ =

```{r}
3 + 1^2/(2*2)
```

```{r}
#sample
11 + 5^2/(2*6)
```

```{r}
#Community
17 + 5^2/(2*12)
```

* What is the chao1 estimate for your sample?  
    - 13.08  
* What is the chao1 estimate for your original total community?  
    - 18.04  

#### Part 4: Alpha-diversity functions in R
We've been doing the above calculations by hand, which is a very good exercise to aid in understanding the math behind these estimates. Not surprisingly, these same calculations can be done with R functions. Since we just have a species table, we will use the `vegan` package. You will need to install this package if you have not done so previously. 

```{r message=FALSE}
library(vegan)
```

First, we must remove the unnecesary data columns and transpose the data so that `vegan` reads it as a species table with species as columns and rows as samples (of which you only have 1).
```{r}
example_data1_diversity = 
  example_data1 %>% 
  select(name, occurences) %>% 
  spread(name, occurences)

example_data1_diversity

Candycount_sample = read.table(file="Candycount_sample.csv", header=TRUE, sep=",", na.strings=c("NAN", "NA", "."))

candy_diversity =
  Candycount_sample %>%
  select(Name, Count) %>%
  spread(Name, Count)

candy_diversity

Candycount_community= read.table(file="Candycount_community.csv", header=TRUE, sep=",", na.strings=c("NAN", "NA", "."))

candy_diversityC =
  Candycount_community %>%
  select(Name, Count) %>%
  spread(Name, Count)

candy_diversityC
```

Then we can calculate the Simpson Reciprocal Index using the `diversity` function.
```{r}
diversity(example_data1_diversity, index="invsimpson")

diversity(candy_diversity, index="invsimpson")

diversity(candy_diversityC, index="invsimpson")
```

And we can calculate the Chao1 richness estimator (and others by default) with the the `specpool` function for extrapolated species richness. This function rounds to the nearest whole number so the value will be slightly different that what you've calculated above.
```{r}
specpool(example_data1_diversity)

specpool(candy_diversity)

specpool(candy_diversityC)
```

In Project 1, you will also see functions for calculating alpha-diversity in the `phyloseq` package since we will be working with data in that form.

For your sample:

* What are the Simpson Reciprocal Indices for your sample and community using the R function?  
    - They are the same as the ones we calculated beforehand (4.401 and 4.881 for sample and community respectively).  
* What are the chao1 estimates for your sample and community using the R function?  
    - They are 11 and 17 for sample and community, respectively. These do not match, as in the csv files created for these, only the total number of cells (undistinguished by colour) are given for the count of each species.  
    + Verify that these values match your previous calculations.  

#### Part 5: Concluding activity
If you are stuck on some of these final questions, reading the [Kunin et al. 2010](https://www.ncbi.nlm.nih.gov/pubmed/19725865) and [Lundin et al. 2012](https://www.ncbi.nlm.nih.gov/pubmed/23760801) papers may provide helpful insights. 

* How does the measure of diversity depend on the definition of species in your samples?  
    - Depending on how we decide to group the candy types (based on characteristics such as colour or shape), a species can be made bigger or smaller. The more characteristics we separate the candies by, the higher the diversity we will have (since more species will be created).    
* Can you think of alternative ways to cluster or bin your data that might change the observed number of species?  
    - We could group the candies based on colour or size, rather than brand.  
* How might different sequencing technologies influence observed diversity in a sample?  
    - Different sequencing technologies have different post-sequencing processing procedures (such as how they are assembled) and quality controls. These can introduce sequencing errors on different levels, such as biases generated from using different models for calculations.  
    
###Project_1  
Team 1: Albert Chang (26234147), Alison Fong (33399149), Karen Lau (16524143), Yaqian Luo (59751503), Jessica Ngo (31837131), Peter (Kiet) Truong (36645133)  

####Abstract  
To study the diversity and biochemical responses of microbial communities in the context of oxygen minimum zones (OMZs), Saanich Inlet was used as a model ecosystem from which water samples were collected at seven major depths spanning the oxycline. A metagenomic study was conducted in which genomic DNA was extracted from the water samples, PCR amplified, assembled into contiguous sequences, and processed into operational taxonomic units (OTUs) and amplicon sequence variants (ASVs) using mothur and QIIME2 pipelines. Based on OTU and ASV results, we chose to focus in on Cyanobacteria as our taxon of interest. Both mothur and QIIME2 data produced alpha diversity values based on Shannon's diversity index that suggest a decrease of Cyanobacterial abundance as depth increases. In addition, Cyanobacterial abundance significantly differs across depth and oxygen concentration according to both mothur and QIIME2 data; specifically, abundance significantly decreases at deeper depths and in environments with lower concentrations of oxygen. Within the Cyanobacteria phylum, there are 15 OTUs and 51 ASVs across all samples. Abundance of five OTUs from the mothur pipeline showed significant changes across both depth and oxygen. In contrast, abundance of none of the ASVs from the QIIME2 pipeline showed significant changes across depth, although 17 ASVs showed significant changes across oxygen concentrations. An increase of Cyanobacterial abundance at shallow depths may be explained in part by their ability to absorb red and orange light at the upper boundaries of the water column. This is also supported by the significant change in oxygen and chlorophyll A concentrations across the depth profile. Another explanation for low Cyanobacterial abundance at lower depths may be due to changes in temperature; where temperature drops below 10^o^C at 100m, Cyanobacterial growth stops completely.To study the diversity and biochemical responses of microbial communities in the context of oxygen minimum zones (OMZs), Saanich Inlet was used as a model ecosystem from which water samples were collected at seven major depths spanning the oxycline. A metagenomic study was conducted in which genomic DNA was extracted from the water samples, PCR amplified, assembled into contiguous sequences, and processed into operational taxonomic units (OTUs) and amplicon sequence variants (ASVs) using mothur and QIIME2 pipelines. Based on OTU and ASV results, we chose to focus in on Cyanobacteria as our taxon of interest. Both mothur and QIIME2 data produced alpha diversity values based on Shannon's diversity index that suggest a decrease of Cyanobacterial abundance as depth increases. In addition, Cyanobacterial abundance significantly differs across depth and oxygen concentration according to both mothur and QIIME2 data; specifically, abundance significantly decreases at deeper depths and in environments with lower concentrations of oxygen. Within the Cyanobacteria phylum, there are 15 OTUs and 51 ASVs across all samples. Abundance of five OTUs from the mothur pipeline showed significant changes across both depth and oxygen. In contrast, abundance of none of the ASVs from the QIIME2 pipeline showed significant changes across depth, although 17 ASVs showed significant changes across oxygen concentrations. An increase of Cyanobacterial abundance at shallow depths may be explained in part by their ability to absorb red and orange light at the upper boundaries of the water column. This is also supported by the significant change in oxygen and chlorophyll A concentrations across the depth profile. Another explanation for low Cyanobacterial abundance at lower depths may be due to changes in temperature; where temperature drops below 10^o^C at 100m, Cyanobacterial growth stops completely.  

####Introduction  
Oxygen minimum zones (OMZs) are areas in the ocean where dissolved oxygen concentrations fall below 20 $\mu$M (1). Due to temperature increases and other effects caused by global warming, OMZs are expanding at a notable rate. Saanich Inlet, a seasonally anoxic fjord off the coast of British Columbia, is a model ecosystem for studying the diversity and biochemical responses of microbial communities to the hypoxic environments commonly observed in OMZs (1, 2). In particular, Saanich Inlet has been used to model the metabolic coupling and symbiotic interactions that occur in OMZs (3).  The inlet undergoes recurring cycles of water column stratification and deep water renewal, rendering it a model ecosystem for studying microbial responses to changes in ocean deoxygenation levels (4). Increased levels of primary productivity in ocean surfaces during the spring season, as well as the limited mixing which occurs between the basin and surface waters both result in the development of an anoxic body of water with increasing depth in the Inlet (2). These anoxic regions become populated with chemolithoautotrophs, and eventually lead to a decrease in aerobically respiring organisms found deeper within these zones. Past studies have demonstrated that these kinds of metabolic shifts generally lead to a loss of nitrogen along with the production of greenhouse gases, most notably methane (CH~4~) and nitrous oxide (N~2~O) (1).  

In order to investigate the changes that occur in these OMZs, water samples of various depths were collected from Saanich Inlet. Genomic DNA was extracted from these to conduct a metagenomics study, allowing to overcome the barrier of uncultivability of these samples and enable a more thorough exploration of the relationship which exists between the microbes and their communities based on genetic distribution of metabolic processes (5, 6, 7, 8). The extracted DNA is sequenced to generate raw data, which can then be assembled into contiguous sequences. These contigs generated by amplicon sequencing are then compared to a sequence database to determine the microbial taxa present in the environment at each water depth. This involves processing the sequencing data, and there currently exists two methods for this type of data analysis: operational taxonomic units (OTUs) and amplicon sequence variants (ASVs). OTU based pipelines work based on clustering reads which differ by less than a fixed dissimilarity threshold (9). This allows more data to be kept, although some may not be representative of the actual taxa in the community. On the contrary, ASV based pipelines resolve these sequence variants by inferring biological sequences in the sample prior to amplification and possible sequencing errors, and are able to distinguish variants which differ by even one nucleotide (9). This treats each ASV as individual species, though has the potential to discard more data and bias towards sequences that are less error-prone.  

The objective of this paper is to analyze the data generated from both OTU and ASV pipelines in order to decide which produces more logical inferences, and ultimately determining which pipeline would be preferred to carry out future analysis of collected water samples. The taxonomy of interest which was selected for this comparison was the phylum Cyanobacteria. Cyanobacteria was selected as there are sufficient numbers of OTUs and ASVs to make sound comparisons, but not so much so that computation-wise it would be infeasible.  

####Methods  

#####Sampling  
Samples were obtained on Saanich Inlet Cruise 72 and taken from seven major depths spanning the oxycline: 10, 100, 120, 135, 150, 165 and 200 m. Waters were filtered, and genomic DNA was extracted. Further sampling details can be found in (3).  

#####DNA Sequencing  
Samples were PCR amplified using the 515F and 808R primers, then sequenced according to the standard operating protocol on an Illumina MiSeq platform with Phred33 quality scores.  

#####Data Processing
Sequences were processed using either mothur or QIIME2 as follows:  

*Mothur Pipeline*: mothur was used to clean-up the data. In brief, paired end reads were combined into contigs using their overlapping regions. Low quality sequences, useless sequence data, chimeric sequences and singletons were removed. OTUs were then determined at 97% similarity. OTUs were classified using the SILVA databases, and the taxonomies for each OTU were condensed. The OTU table, taxonomy data and sample metadata were subsequently cleaned up and combined into a phyloseq object.  

*QIIME2 Pipeline*: Demultiplexed sequences were imported into QIIME as manifest reads. QIIME was used to clean up the data along with ASV determination in one step. Sequence quality was visually evaluated, and sequence quality trimming was conducted. All other trimming/filtering parameters were left as default. ASV determination was completed using the Dada2 protocol. ASV classification was completed using the Silva version 119 database at 99% similarity. The ASV table, taxonomy data and sample metadata were subsequently cleaned up and combined into a phyloseq object.  

#####Data Analysis   
The aforementioned phyloseq objects were imported into R version 3.4.3 (Windows) or 1.1.383 (Mac). The tidyverse, phyloseq, magrittr, knitr and cowplot packages were loaded and used to complete the data analysis. Data was piped into linear models and ANOVA tests to determine statistical significance at the 95% confidence level.  

######Environment setup and Data Cleaning
```{r, echo=FALSE, "loading packages and data"}
library("tidyverse")
library("phyloseq")
library("magrittr")
library("knitr")
library("cowplot")

load("./mothur_phyloseq.RData")
load("./qiime2_phyloseq.RData")
```

```{r, echo=FALSE}
knitr::opts_chunk$set(error = TRUE)
```

```{r, echo=FALSE, "data cleaning"}
#Mothur data
set.seed(4832)
m.norm = rarefy_even_depth(mothur, sample.size=100000)
m.perc = transform_sample_counts(m.norm, function(x) 100 * x/sum(x))

#Qiime2 data
set.seed(4832)
q.norm = rarefy_even_depth(qiime2, sample.size=100000)
q.perc = transform_sample_counts(q.norm, function(x) 100 * x/sum(x))

```

####Results  

1. How does microbial community structure change with depth and oxygen concentration?  

**Alpha-diversity comparison:**   
*Mothur*  
As shown in Figure 1, alpha-diversity by Shannon's diversity index based on mothur shows an overall decreasing trend as depth increases. Specifically, a stable trend is observed from depths of 0-100m, starts decreasing at 100-150m and stabilizes at 150-200m. Figure 1 also shows an overall increasing trend as oxygen increases. Specifically, Shannon's diversity index increases from 2.3-4.25 at oxygen concentrations of 0-40$\mu$M and decreases from 4.25-3.9 at oxygen concentrations of 40-220µM. It was also observed that Shannon's diversity index is higher in oxic conditions (3.84 ± 0.45) than anoxic conditions (2.39 ± 0.07).  

*QIIME2*  
The trends of Shannon's diversity index across depth and oxygen based on QIIME2 data are similar to those of mothur data. However, QIIME2 pipeline produces higher Shannon's diversity index than mothur does. Shannon's diversity index increases from 2.9-5.2 at oxygen concentrations of 0-40$\mu$M and decreases from 5.2-5.1 at oxygen concentrations of 40-200m. Figure 2 shows that Shannon's diversity index is higher in oxic conditions (4.80 ± 0.43) than anoxic conditions (3.15 ± 0.18).  

######Alpha-diversity of mothur data
```{r, echo=FALSE, "Question 1a_1", fig.width=14, fig.height=10}
#Mothur data
m.alpha = estimate_richness(m.norm, measures = c("Chao1", "Shannon"))
m.meta.alpha = full_join(rownames_to_column(m.alpha), rownames_to_column(data.frame(m.perc@sam_data)), by = "rowname")

#Mother data - Alpha diversity across depth
plot_1=m.meta.alpha %>%
ggplot() +
   geom_point(aes(x=Depth_m, y=Shannon)) +
   geom_smooth(method='auto', aes(x=as.numeric(Depth_m), y=Shannon)) +
   labs(y="Shannon's diversity index", x="Depth (m)")

#Mothur data - Alpha diversity across oxygen
plot_2=m.meta.alpha %>%
ggplot() +
  geom_point(aes(x=O2_uM, y=Shannon)) +
  labs(y="Shannon's diversity index", x="Oxygen (uM)")

#Mothur data - Alpha diversity by oxic/anoxic
plot_3=m.meta.alpha %>%
  mutate(O2_group = ifelse(O2_uM == 0, "anoxic", "oxic")) %>%
ggplot() +
  geom_boxplot(aes(x=O2_group, y=Shannon)) +
  labs(y="Shannon's diversity index", x="Oxygen")

plot_grid(plot_1, plot_2, plot_3, labels=c("A", "B","c"))+
   labs(caption="Figure 1. Alpha-diversity across depth or oxygen using mothur data. A) Alpha-diversity across depth using mothur data.
        B) Alpha-diversity across oxygen using mothur data. C) Alpha-diversity by oxic/anoxic using mothur data")+
  theme(plot.caption = element_text(size=rel(1.2)))

# Average and standard deviation of mothur by oxic/anoxic
m.oxygen=m.meta.alpha %>%
  mutate(O2_group = ifelse(O2_uM == 0, "anoxic", "oxic")) 

m.anoxic=m.oxygen%>%
  filter(O2_group=="anoxic")
  
ave.m.anoxic=mean(m.anoxic$Shannon)
st.m.anoxic=sd(m.anoxic$Shannon)

m.oxic=m.oxygen%>%
  filter(O2_group=="oxic")

ave.m.oxic=mean(m.oxic$Shannon)
st.m.oxic=sd(m.oxic$Shannon)

m.avg_std = data.frame(
  Statistic=c("Average","Standard deviation"),
  Oxic=c(ave.m.oxic, st.m.oxic),
  Anoxic=c(ave.m.anoxic, st.m.anoxic)
)

kable(m.avg_std,captio="Table 1. Average and standard deviation of alpha-diversity by oxic/anoxic with mothur data", align = "c")

```

######Alpha-diversity of QIIME2 data
``` {r, echo=FALSE, "Question 1a_2", fig.width=14, fig.height=10}
#Qiime2 data
q.alpha = estimate_richness(q.norm, measures = c("Chao1", "Shannon"))
q.meta.alpha = full_join(rownames_to_column(q.alpha), rownames_to_column(data.frame(q.perc@sam_data)), by = "rowname")

#Qiime 2 data - Alpha diversity across depth
plot_4=q.meta.alpha %>%
 
ggplot() +
   geom_point(aes(x=Depth_m, y=Shannon)) +
   geom_smooth(method='auto', aes(x=as.numeric(Depth_m), y=Shannon)) +
   labs(y="Shannon's diversity index", x="Depth (m)")
 
#Qiime2 data - Alpha-diversity across oxygen
plot_5=q.meta.alpha %>%
ggplot() +
  geom_point(aes(x=O2_uM, y=Shannon)) +
  labs(y="Shannon's diversity index", x="Oxygen (uM)") 
 
#Qiime2 data - Alpha-diversity by oxic/anoxic
plot_6=q.meta.alpha %>%
  mutate(O2_group = ifelse(O2_uM == 0, "anoxic", "oxic")) %>%
ggplot() +
  geom_boxplot(aes(x=O2_group, y=Shannon)) +
  labs(y="Shannon's diversity index", x="Oxygen")
  
plot_grid(plot_4, plot_5, plot_6, labels=c("A", "B","c"))+
   labs(caption="Figure 2. Alpha-diversity across depth or oxygen using QIIME2 data. A) Alpha-diversity across depth using QIIME2 data. 
        B) Alpha-diversity across oxygen using QIIME2 data. C) Alpha-diversity by oxic/anoxic using QIIME2 data")+
  theme(plot.caption = element_text(size=rel(1.2)))

# Average and standard deviation of quiime2 by oxic/anoxic
q.oxygen=q.meta.alpha %>%
  mutate(O2_group = ifelse(O2_uM == 0, "anoxic", "oxic")) 

q.anoxic=q.oxygen%>%
  filter(O2_group=="anoxic")

ave.q.anoxic=mean(q.anoxic$Shannon)
sd.q.anoxic=sd(q.anoxic$Shannon)

q.oxic=q.oxygen%>%
  filter(O2_group=="oxic")

ave.q.oxic=mean(q.oxic$Shannon)
sd.q.oxic=sd(q.oxic$Shannon)

q.avg_std = data.frame(
  Statistic=c("Average","Standard deviation"),
  Oxic=c(ave.q.oxic, sd.q.oxic),
  Anoxic=c(ave.q.anoxic, sd.q.anoxic)
)

kable(q.avg_std,captio="Table 2. Average and standard deviation of alpha-diversity by oxic/anoxic with QIIME2 data", align = "c")

```

**Taxa presence and abundance:**  
*Mothur*  
31 taxa in the phylum level are detected by mothur pipeline (Figure 3). These taxa, however, have abundance at different magnitudes. Among of them, Proteobacteria is the most predominant phylum in all samples with the highest average abundance over 75. On the contrary, phylum Peregrinlbacteria has abundance no larger than 0.001 in the seven samples (Figure 3). Additionally, different taxa have distinct changes in abundance across depth. For instance, both Thaumarchaeota and Verrucomicrobia reach their maximum abundance at depth of 100m and and gradually decrease in abundance until 200m, while Latescibacteria and Fibrobacteres are almost undetectable in shallow water and their abundances increase dramatically at depth of 200m.  

*QIIME2*  
QIIME2 pipeline detects 29 known taxa and unknown taxa in phylum level (Figure 4).  Proteobacteria is still the most abundant phylum across samples. Taxa Chlorobi and Candidate division OP3 have the smallest abundance no larger than 0.004. Different pipelines may result in different changes in abundance for the common taxa shared by mothur and QIIME2 data. Although the change patterns of phylum Actinobacteria in abundance across depth are the same in both datasets, Chloroflexi abundance increases gradually with depth in QIIME2 data different from its change pattern in mothur data, in which abundance declines at depth from 100m to 135m and increases gradually until 200m.  

```{r, echo=FALSE, "Question 1b_1", fig.width=19, fig.height=15}
#Mothur data - Phyla across samples
m.perc %>%
plot_bar() +
  geom_bar(aes(fill=Phylum), stat="identity") +
  facet_wrap(~Phylum, scales="free_y")+
  theme(legend.position="none")+
  labs(caption="Figure 3. Taxa within phylum level across samples using mothur data")+
  theme(plot.caption = element_text(hjust=0.5, size=rel(1.7)))
```

```{r, echo=FALSE, "Question 1b_2", fig.width=18, fig.height=13}
#Qiime2 data - Phyla across samples
q.perc %>%
plot_bar() +
  geom_bar(aes(fill=Phylum), stat="identity") +
  facet_wrap(~Phylum, scales="free_y")+
  theme(legend.position="none")+
  labs(caption="Figure 4. Taxa within phylum level across samples using qiime2 data")+
  theme(plot.caption = element_text(hjust=0.5, size=rel(1.7)))
```


2. Does your taxon of interest significantly differ in abundance with depth and/or oxygen concentration?  

*Mothur*  
The difference in cyanobacteria abundance within depth or oxygen was estimated by the linear model using mothur processed data. The statistical results show that abundance of cyanobacteria is significantly different with depth (p = 0.01263) and oxygen (p = 0.00012). However linear models in Figure 5 indicate completely distinct trends of cyanobacteria abundance across depth and oxygen, where there is a decrease in abundance as depth increases and an increase in abundance as oxygen concentrations increases respectively.  

*QIIME2*  
For data processed by QIIME2 pipeline, ANOVA tests indicate cyanobacteria abundance in the seven samples has significantly difference across depth (p = 0.014) and oxygen (p = 0.013).The linear models in Figure 6 reveal that cyanobacteria abundance decreases at deeper water or in the environment with insufficient oxygen.  

```{r, echo=FALSE, "Question 2a", fig.width=10, fig.height=4}
#Calculating significance with linear model for depth
m.norm %>% 
  subset_taxa(Phylum=="Cyanobacteria") %>% 
  tax_glom(taxrank = 'Phylum') %>%
  psmelt() %>%
  
  lm(Abundance ~ Depth_m, .) %>% 
  summary()

#Plot to go along with linear model above
plot_7=m.perc %>% 
  subset_taxa(Phylum=="Cyanobacteria") %>% 
  psmelt() %>% 
  group_by(Sample) %>% 
  summarize(Abundance_sum=sum(Abundance), Depth_m=mean(Depth_m)) %>% 
  
  ggplot() +
  geom_point(aes(x=Depth_m, y=Abundance_sum)) +
  geom_smooth(method='lm', aes(x=as.numeric(Depth_m), y=Abundance_sum)) 

#Calculating significance with linear model for oxygen
m.norm %>% 
  subset_taxa(Phylum=="Cyanobacteria") %>% 
  tax_glom(taxrank = 'Phylum') %>% 
  psmelt() %>% 
  
  lm(Abundance ~ O2_uM, .) %>% 
  summary()

#Plot to go along with linear model above
plot_8=m.perc %>% 
  subset_taxa(Phylum=="Cyanobacteria") %>% 
  psmelt() %>% 
  group_by(Sample) %>% 
  summarize(Abundance_sum=sum(Abundance), O2_uM=mean(O2_uM)) %>% 
  
  ggplot() +
  geom_point(aes(x=O2_uM, y=Abundance_sum)) +
  geom_smooth(method='lm', aes(x=as.numeric(O2_uM), y=Abundance_sum)) 

plot_grid(plot_7, plot_8, labels=c("A", "B"))+
  labs(caption="Figure 5. Abundance of Cyanobacteria across depth/oxygen concentrations using mothur OTUs. 
       A) Abundance of Cyanobacteria across depth. B) Abundance of Cyanobacteria across oxygen concentrations.")+
  theme(plot.caption = element_text(hjust=0.5,size=rel(0.8)))
```


```{r, echo=FALSE, "Question 2b", fig.width=10, fig.height=4}

#Calculating significance with linear model for depth
q.norm %>%
  subset_taxa(Phylum=="D_1__Cyanobacteria") %>%
  tax_glom(taxrank = 'Phylum') %>%
  psmelt() %>%
  
  lm(Abundance ~ Depth_m, .) %>%
  summary()

#Plot to go along with linear model above
plot_9=q.perc %>%
  subset_taxa(Phylum=="D_1__Cyanobacteria") %>%
  psmelt() %>%
  group_by(Sample) %>%
  summarize(Abundance_sum=sum(Abundance), Depth_m=mean(Depth_m)) %>%
 
ggplot() +
  geom_point(aes(x=Depth_m, y=Abundance_sum)) +
  geom_smooth(method='lm', aes(x=as.numeric(Depth_m), y=Abundance_sum)) 

#Calculating significance with linear model for oxygen
q.norm %>%
  subset_taxa(Phylum=="D_1__Cyanobacteria") %>%
  tax_glom(taxrank = 'Phylum') %>%
  psmelt() %>%
  lm(Abundance ~ O2_uM, .) %>%
  summary()

#Plot to go along with linear model above
plot_10=q.perc %>%
  subset_taxa(Phylum=="D_1__Cyanobacteria") %>%
  psmelt() %>%
  group_by(Sample) %>%
  summarize(Abundance_sum=sum(Abundance), O2_uM=mean(O2_uM)) %>%
 
ggplot() +
  geom_point(aes(x=O2_uM, y=Abundance_sum)) +
  geom_smooth(method='lm', aes(x=as.numeric(O2_uM), y=Abundance_sum)) 

plot_grid(plot_9, plot_10, labels=c("A", "B"))+
  labs(caption="Figure 6. Abundance of Cyanobacteria across depth/oxygen concentrations using QIIME2 ASVs. 
       A) Abundance of Cyanobacteria across depth. B) Abundance of Cyanobacteria across oxygen concentrations.")+
  theme(plot.caption = element_text(hjust=0.5,size=rel(0.8)))
```

3. Within your taxon, what is the richness (number of OTUs/ASVs)?  

*Mothur*  
Across all samples, there are 15 OTUs within cyanobacteria phylum. Table 3 presents the numbers of OTUs within cyanobacteria phylum for each sample. Most of samples contain  3-5 OTUs within cyanobacteria, except Saanich_120 and Saanich_200 which only have 1 and 0 cyanobacteria OTUs respectively.  

*QIIME2*  
There are 51 ASVs within cyanobacteria phylum across all samples. The number of ASVs within cyanobacteria phylum for each sample is shown in Table 2. Saanich_010, Saanich 120, and Saanich_135  have relatively high ASV number equal to or over 15. However, it is important to note that there are no singletons within the ASV dataset, and the function used to estimate richness, "estimate_richness" from the phyloseq library is highly dependent on the number of singletons, and warns of unreliable or wrong results in the absence of singletons in the data.  


######Table showing richness of OTUs using mothur data and ASVs using QIIME2 data  
```{r, echo=FALSE, "Question 3a"}
m.norm %>% 
  subset_taxa(Phylum=="Cyanobacteria")

mothur_otu=m.norm %>% subset_taxa(Phylum=="Cyanobacteria") %>% estimate_richness(measures = c("Observed"))

q.norm %>% 
  subset_taxa(Phylum=="D_1__Cyanobacteria")

qiime_asv=q.norm %>% subset_taxa(Phylum=="D_1__Cyanobacteria") %>%estimate_richness(measures = c("Observed"))

sam_data=data.frame(m.norm@sam_data)

Depth=sam_data%>%select("Depth_m")

otu_asv=cbind(Depth, mothur_otu, qiime_asv)
colnames(otu_asv)=c("Depth_m","OTU","ASV")
kable(otu_asv, caption="Table 3. OTUs/ASVs across depth",align = "c")

```

4. Do the abundances of OTUs/ASVs within your taxon of interest change significantly with depth and/or oxygen concentration?  

Using the linear model for statistical interpretation, after correcting the p-value for multiple comparisons, the abundance of  OTUs 0189, 0658, 1104, 3852, and 4312 from the mothur pipeline within the cyanobacteria phylum changed significantly with both depth and oxygen. Interestingly, the abundance of ASVs in the QIIME2 pipeline did not have any significant changes across the depth profiles after correcting for the p-value. However, there were 17 ASVs that had a significant abundance change with respect to oxygen concentrations.  

######Mothur
```{r, echo=FALSE, "Question 4a",fig.width=10,fig.height=7}
#filtered table 
m.Cyano=m.norm %>% subset_taxa(Phylum=="Cyanobacteria") 
m.filtered=data.frame(m.Cyano@tax_table)

#General linear model for each OTU across depth
otu_stats = data.frame("Estimate" = numeric(0), "Std. Error"= numeric(0),"t value"= numeric(0),"P_value"= numeric(0))
for (otu in row.names(m.filtered)){
  require(tidyverse)
  require(knitr)
    linear_fit = m.norm %>% 
    psmelt() %>% 
    filter(OTU==otu) %>% 
    lm(Abundance ~ Depth_m, .) %>% 
    summary()
  otu_data = linear_fit$coefficients["Depth_m",]
  otu_stats <- rbind(otu_stats, otu_data)
}

colnames(otu_stats)<- (c("Estimate", "Std. Error","t value","P_value"))
otu_depth=otu_stats%>%
  mutate(Adjusted_P=p.adjust(P_value, method="fdr"))
row.names(otu_depth) <- row.names(m.filtered)
otu_depth_filter=otu_depth[which(otu_depth[,"Adjusted_P"]<=0.05),]
kable(otu_depth_filter,caption="Table 4. Correlation data of Cyanobacteria OTUs with significant differences across depth using mothur data",align = "c")

#Calculating abundances of OTUs across depth
m.perc %>%
  subset_taxa(Phylum=="Cyanobacteria") %>%
  psmelt() %>%
 
ggplot() +
  geom_point(aes(x=Depth_m, y=Abundance)) +
  geom_smooth(method='lm', aes(x=Depth_m, y=Abundance)) +
  facet_wrap(~OTU, scales="free_y") +
  labs(caption="Figure 7. Abundance of OTUs within Phylum Cyanobacteria across depth using mothur data")+
  theme(plot.caption = element_text(hjust=0.5, size=rel(0.9)))

#General linear model for each OTU across oxygen
otu_stats_O2 = data.frame("Estimate" = numeric(0), "Std. Error"= numeric(0),"t value"= numeric(0),"P_value"= numeric(0))
for (otu in row.names(m.filtered)){
  require(tidyverse)
  require(knitr)
    linear_fit = m.norm %>% 
    psmelt() %>% 
    filter(OTU==otu) %>% 
    lm(Abundance ~ O2_uM, .) %>% 
    summary()
  otu_data_O2 = linear_fit$coefficients["O2_uM",]
  otu_stats_O2 <- rbind(otu_stats_O2, otu_data_O2)
}

colnames(otu_stats_O2)<- (c("Estimate", "Std. Error","t value","P_value"))
otu_oxygen=otu_stats_O2%>%
  mutate(Adjusted_P=p.adjust(P_value, method="fdr"))
row.names(otu_oxygen) <- row.names(m.filtered)
otu_oxygen_filter=otu_oxygen[which(otu_oxygen[,"Adjusted_P"]<=0.05),]
kable(otu_oxygen_filter,caption="Table 5. Correlation data of Cyanobacteria OTUs with significant differences across oxygen using mothur data", align = "c")

#Calculating abundances of OTUs across oxygen
m.perc %>%
  subset_taxa(Phylum=="Cyanobacteria") %>%
  psmelt() %>%
 
ggplot() +
  geom_point(aes(x=O2_uM, y=Abundance)) +
  geom_smooth(method='lm', aes(x=O2_uM, y=Abundance)) +
  facet_wrap(~OTU, scales="free_y") +
  labs(caption="Figure 8. Abundance of OTUs within Cyanobacteria across oxygen using mothur data")+
  theme(plot.caption = element_text(hjust=0.5, size=rel(0.9)))

```

######QIIME2
```{r, echo=FALSE, "Question 4b_1"}
#filtered table 
q.Cyano=q.norm %>% subset_taxa(Phylum=="D_1__Cyanobacteria") 
q.filtered=data.frame(q.Cyano@tax_table)

#General linear model for each ASV across depth
asv_stats = data.frame("Estimate" = numeric(0), "Std. Error"= numeric(0),"t value"= numeric(0),"P_value"= numeric(0))
for (asv in row.names(q.filtered)){
  require(tidyverse)
  require(knitr)
    linear_fit = q.norm %>% 
    psmelt() %>% 
    filter(OTU==asv) %>% 
    lm(Abundance ~ Depth_m, .) %>% 
    summary()
  asv_data = linear_fit$coefficients["Depth_m",]
  asv_stats <- rbind(asv_stats, asv_data)
}

colnames(asv_stats)<- (c("Estimate", "Std. Error","t value","P_value"))
asv_depth=asv_stats%>%
  mutate(Adjusted_P=p.adjust(P_value, method="fdr"))
row.names(asv_depth) <- row.names(q.filtered)

asv_depth_filter=asv_depth[which(asv_depth[,"Adjusted_P"]<=0.05),]
z=nrow(asv_depth_filter)

if (z!=0) {
  kable(asv_depth_filter,caption="Correlation data of Cyanobacteria ASVs with significant differences across depth using QIIME2 data",align = "c")
}else{
    print("None of ASV has significantly different abundance acrossing depth with QIIME2 data")}
```

```{r, echo=FALSE, "Question 4b_2", fig.width=15, fig.height=10}
#Calculating abundances of ASVs across depth
q.perc %>%
  subset_taxa(Phylum=="D_1__Cyanobacteria") %>%
  psmelt() %>%
 
ggplot() +
  geom_point(aes(x=Depth_m, y=Abundance)) +
  geom_smooth(method='lm', aes(x=Depth_m, y=Abundance)) +
  facet_wrap(~OTU, scales="free_y") +
  labs(caption="Figure 9. Abundance of ASVs within cyanobacteria across depth using QIIME2 data")+
  theme(plot.caption = element_text(hjust=0.5, size=rel(1.3)))
```

```{r, echo=FALSE, "Question 4b_3"}
#General linear model for each ASV across oxygen
asv_stats_O2 = data.frame("Estimate" = numeric(0), "Std. Error"= numeric(0),"t value"= numeric(0),"P_value"= numeric(0))
for (asv in row.names(q.filtered)){
  require(tidyverse)
  require(knitr)
    linear_fit = q.norm %>% 
    psmelt() %>% 
    filter(OTU==asv) %>% 
    lm(Abundance ~ O2_uM, .) %>% 
    summary()
  asv_data_O2 = linear_fit$coefficients["O2_uM",]
  asv_stats_O2 <- rbind(asv_stats_O2, asv_data_O2)
}

colnames(asv_stats_O2)<- (c("Estimate", "Std. Error","t value","P_value"))
asv_oxygen=asv_stats_O2%>%
  mutate(Adjusted_P=p.adjust(P_value, method="fdr"))
row.names(asv_oxygen) <- row.names(q.filtered)
asv_oxygen_filter=asv_oxygen[which(asv_oxygen[,"Adjusted_P"]<=0.05),]
kable(asv_oxygen_filter,caption="Table 6. Correlation data of Cyanobacteria ASVs with significant differences across oxygen using QIIME2 data", align = "c")
```

```{r, echo=FALSE, "Question 4b_4", fig.width=16, fig.height=10}
#Calculating abundance of ASVs across oxygen
q.perc %>%
  subset_taxa(Phylum=="D_1__Cyanobacteria") %>%
  psmelt() %>%
 
ggplot() +
  geom_point(aes(x=O2_uM, y=Abundance)) +
  geom_smooth(method='lm', aes(x=O2_uM, y=Abundance)) +
  facet_wrap(~OTU, scales="free_y") +
  labs(caption="Figure 10. Abundance of ASVs within Cyanobacteria across oxygen using QIIME2 data")+
  theme(plot.caption = element_text(hjust=0.5, size=rel(1.3)))
```

5. Are the answers to the above the same using mothur and QIIME2 processed data?  

In terms of differences between the two pipelines, the overall trend between both datasets were similar, however the statistical interpretations of the datasets varied. For starters, the Shannon diversity for the whole microbial community processed with mothur was generally smaller than that estimated by QIIME 2 processed data. Additionally, ANOVA tests indicated that the Shannon diversity had no significant change across depth from the mothur pipeline (p = 0.054). In contrast, there was statistical significance with the Shannon diversity across depth from the QIIME2 dataset (p = 0.022). Within the Cyanobacterial taxon, there were 51 ASVs calculated from the QIIME2 pipeline compared to 15 OTUs from the Mothur pipeline. Interestingly, while there is a difference in the richness between both datasets, 5 of the OTUs had a significantly different abundance with respect to depth and oxygen profiles, while none of the ASVs had a significant change in abundance across depth and 17 ASVs which had a significant difference in abundance across oxygen concentrations. Despite all these variations, the Cyanobacterial taxon itself had a significant difference in abundance with respect to depth and oxygen profiles along the water column across both pipelines.

###### ANOVA on alpha-diversity of mothur data across depth 
```{r,echo=FALSE}
#Mothur data - ANOVA test on Alpha diversity across depth
summary(aov(Shannon ~ Depth_m, data = m.meta.alpha))
```
###### ANOVA on alpha-diversity of QIIME2 data across depth
```{r, echo=FALSE}
#Qiime2 data - ANOVA test on Alpha diversity across depth
summary(aov(Shannon ~ Depth_m, data = q.meta.alpha))
```

####Discussion

The significant differences for Cyanobacteria abundance across depth and oxygen may be caused by photosynthesis and water temperature. Bacteria of the phylum Cyanobacteria obtain their energy through photosynthesis. These phototrophs are characterized by phycocyanin, a bluish pigment, which functions as an auxiliary light-harvesting protein complex to chlorophyll. Phycocyanin absorbs orange and red light at approximately 620nm and fluoresces at about 650nm depending on the species (10). This is particularly interesting because orange and red light are typically absorbed within the first 50m of a water column (11). From our data in Fig 5 & 6, it was found that there are significant differences in the abundance of Cyanobacteria across the Saanich Inlet depth profiles in both the mothur and QIIME2 pipelines. It is likely that this significance exists as a result of Cyanobacteria thriving at the upper boundaries of the water column, where they are still able to absorb red and orange light that is essential for Cyanobacterial photosynthesis. Furthermore, these findings are supported by the oxygen and fluorescence profiles across the water column (Fig 11). There is a significant difference in the concentration of oxygen and chlorophyll A across depth profiles. Higher concentrations of chlorophyll A and oxygen were found within the top 50m, which indicates increased photosynthetic activity. Moreover, there was a significant difference in the abundance of Cyanobacteria across oxygen concentrations and chlorophyll A concentrations for both pipelines; high oxygen and chlorophyll A concentrations were associated with a larger abundance. With Cyanobacterial photosynthesis contributing a substantial proportion of oxygen to Earth's atmosphere, it is not surprising that a high abundance of Cyanobacteria is associated with a high concentration of oxygen and chlorophyll A, and a shallow depth. Moreover, it has been reported that the growth rate of cyanobacteria is significantly influenced by temperature. Cyanobacteria were observed to have a lower growth rate at colder temperatures. For marine cyanobacteria, the optimal growth temperature range is 20 - 27.5^o^C; at these temperatures, cyanobacteria grow at a rate of 0.8 d^-1^  (12). When the temperature dropped to approximately 15^o^C, the growth rate of cyanobacteria slowed to 0.22 d^-1^. Interestingly, when temperature dropped below 10^o^C, cyanobacterial growth rates came to a complete stop (13). Therefore, in our study, the decline of cyanobacteria abundance with depth may at least partly attributed to the decreasing temperature. According temperature data for each sample, the temperature is close to 13^o^C at 10m, and decreases to about 9^o^C when at depths below 100m. Hence, the growth rate of cyanobacteria at depths below 100m is substantially slower than the growth rate at the water's surface. This leads to a lower abundance at lower depths.  

Implications of potential differences in pipelines for microbial ecology make it difficult to make conclusive statements in research and discovery, since we become unable to differentiate actual biological differences seen and differences due to a particular pipeline being used. This could also suggest that one pipeline is more suited to the dataset. In fact, this difference could also be exploited, and manipulated so that a pipeline is selected based on the results that it gives, rather than the more appropriate pipeline for the given dataset.  

In the context of this project, the main difference between the two pipelines is based on whether the pipeline produces OTUs (mothur) or ASVs (QIIME2). Both of them use different clustering algorithms to determine "true" sequences, and as a result, there are usually far more ASVs in the QIIME2 pipeline than OTUs created with the mothur pipeline. Therefore, when doing downstream analysis of the pipeline results, this may be one of the reasons why there is a large disparity in between the numbers of sequences and quality of the sequences produced even when using the same initial data.  

However, when it came to counting abundances within our taxon, cyanobacteria, there was a far lower abundance seen in qiime2 results than mothur results which cannot be explained by having more numerous ASVs than OTUs. Interestingly, when running the estimate_richness function on our qiime2 data for determining abundance, this resulted in the warning that our data provided did not have any singletons (supposedly in the output), and that results are probably unreliable or wrong. This error did not occur with running this function on mothur data.
Further analysis of why this error is seen with qiime2 data should be done before fully trusting the results of this function with qiime2 data.  

In subsequent analyses with these two pipelines, perhaps a more in-depth analysis of each function available with the phyloseq package should be tested. A dataset of a well-studied and known community could be used so that the output of the functions can be compared with respect to the pipeline used. Comparisons can be made for the outputs of each pipeline, and results from the mothur and QIIME2 pipelines can be assessed with reference to the expected results. Use cases should also be considered, and standard use cases for either pipeline should be indicated so exploitation of pipelines for favourable results does not occur. Mothur may be more suited to a particular dataset, while QIIME2 could be more appropriate for another dataset that should be used with a "denoising" algorithm.  

Future directions for this project could possibly involve the analysis of water samples from other OMZs at various depths, along with observation of the Cyanobacteria data present in those samples to see if the relationships are exhibited between oxygen, nitrogen, and the phyla population as with this study. These can also be analyzed once again with both mothur and QIIME2 for further comparison between the pipelines.  

```{r, echo=FALSE, fig.width=10, fig.height=7}
# Geochemical parameters across depth
sam_data=data.frame(m.norm@sam_data)
sam_data1=sam_data%>%mutate(CH4_uM=CH4_nM/1000)
nutrient=sam_data1 %>% select(Depth_m,O2_uM,PO4_uM,SiO2_uM,NO3_uM,NH4_uM,NO2_uM,H2S_uM,N2O_nM,CH4_uM,Temperature_C, Fluorescence_mgm_3) %>% gather(nutrient,uM,O2_uM,PO4_uM,SiO2_uM,NO3_uM,NH4_uM,NO2_uM,H2S_uM,N2O_nM,CH4_uM, Temperature_C, Fluorescence_mgm_3)
library(ggplot2)
ggplot(nutrient,aes(x=Depth_m,y=uM))+
  geom_line()+
  geom_point()+
  ylab("Concentration or degree or mgm3")+
  facet_wrap(~nutrient,scales="free_y")+
  labs(caption="Figure 11. Geochemical parameters across depth")+
  theme(plot.caption = element_text(hjust=0.5, size=rel(0.8)))
```


####References

1. Walsh DA, Zaikova E, Howes CG, Song YC, Wright JJ, Tringe SG, Hallam SJ. 2009. Metagenome of a versatile chemolithoautotroph from expanding oceanic dead zones. Science 326(5952): 578-582.  
2. Torres-Beltrán M, Hawley AK, Capelle D, Zaikova E, Walsh DA., Mueller A, Finke J. 2017. A compendium of geochemical information from the Saanich Inlet water column. Nature scientific data 4(170159).  
3. Hawley AK, Torres-Beltrán M, Zaikova E, Walsh DA, Mueller A, Scofield M, Kheirandish S, Payne C, Pakhomova L, Bhatia M, Shevchek O, Gies EA, Fairley D, Malfatii SA, Norbeck AD, Brewer HM, Pasa-Tolic, L, del Rio TG, Suttle CA, Trige S, Hallam SJ. Data Descriptor: A compendium of multi-omic sequence information from the Saanich Inlet water column. Nature scientific data 4(170160).  
4. Hallam SJ, Torres-Beltrán M, Hawley AK. Comment: Monitoring microbial responses to ocean deoxygenation in a model oxygen minimum zone. Nature scientific data 4(170158).  
5. National Research Council. 2007. The new science of metagenomics: revealing the secrets of our microbial planet. National Academies Press, Washington, DC.  
6. Bowers RM, Kyrpides NC, Stepanauskas R, Harmon-Smith M, Doud D, Reddy TBK, Tringe SG. 2017. Minimum information about a single amplified genome (MISAG) and a metagenome-assembled genome (MIMAG) of bacteria and archaea. Nature Biotechnology 35: 725-731.  
7. Falkowski PG, Fenchel T, Delong EF. 2008. The microbial engines that drive Earth's biogeochemical cycles. Science 320(5879): 1034-1039.  
8. Wooley JC, Godzik A, Friedberg I. 2010. A primer on metagenomics. PLoS computational biology 6(2), e1000667.  
9. Callahan BJ, McMurdie PJ, Holmes SP. 2017. Exact sequence variants should replace operational taxonomic units in marker-gene data analysis. The ISME journal 11(12): 2639-2643.  
10. Simis SG, Huot Y, Babin M, Seppala J, and Metsamaa L. 2012. Optimization of variable fluorescence measurements of phytoplankton communities with cyanobacteria. Photosynthesis research 112(1): 13-30.  
11. Light in the ocean (n.d.) [Online]. Link: https://manoa.hawaii.edu/exploringourfluidearth/physical/ocean-depths/light-ocean.  
12. Boyd PW,  RynearsonTA, Armstrong EA, Fu F, Hayashi K, Hu Z, Hutchins DA, Kudela RM, Litchman E, Mulholland MR, Passow U, Strzepek RF,  Whittaker KA,  Yu E,  and Thomas MK. (2013). Marine phytoplankton temperature versus growth responses from polar to tropical waters - outcome of a scientific community-wide study. PLoS ONE 8(5): e63091.  
13. Berg M, Sutula M. 2015. Factors affecting the growth of cyanobacteria with special emphasis on the Sacramento-San Joaquin Delta. Southern California Coastal Water Research Project Technical Report 869.  

###Evidence worksheet_05 "Extensive mosaic structure"  

####Part 1: Learning objectives:   

Evaluate the concept of microbial species based on environmental surveys and cultivation studies.  

Explain the relationship between microdiversity, genomic diversity and metabolic potential  

Comment on the forces mediating divergence and cohesion in natural microbial communities  

####General Questions:  

* What were the main questions being asked?  

    - Understanding the pathogenecity and eveolutionary diversity of E. coli  
    - How do different E. coli strains differ at genomic scale and at physiological scale?  
    - Is there a physiological difference of just a pathological difference?  

* What were the primary methodological approaches used?  
    - isolated CFT073  
    - PCR and primer walking experiment  
    - obtain whole-genome libraries (shotgun and assemble DNA)  
    - sequenced random clones  
    - Sanger sequencing  
    - Annotated genome sequence  
    - defined ORFs and Blast predicted proteins  

* Summarize the main results or findings.  
    - 12 distinct types of fimbriae gene in genome of CFT073  
    - only 39.2% of their combined (nonredundant) set of proteins are common; thus, only 39.2% of genome define them as E. coli; only 3% divergence in the 16S level  
    - codon variation; GC content; proximity to tRNA (relevant to gene transfer)
    - difference in disease potential between 0157:H7 and CFT073 is the absence of genes for type III secretion system of phage and plasmid-encoded toxins  
    - CFT073 genome rich in genes that encode fimbrial adhesins, autotransporters, iron-sequetration systems and phase-switch recombinases  
    - many islands acquired by different horizontal transfer events in each strain  
    - 2/3 of island genes shared by EDL933 and CFT073 have unknown functions or are associated with phage or insertion sequence elements  

* Do new questions arise from the results?  
    - Why is there limited intragenomic rearrangement in vertical evolution?  
    - What kind of body signaling molecules/features allow pathogenic bacteria to colonize at the specific location?  
    - Cannot use 16S to fingerprint; why are we sequencing E. coli based on 16S?

* Were there any specific challenges or advantages in understanding the paper (e.g. did the authors provide sufficient background information to understand experimental logic, were methods explained adequately, were any specific assumptions made, were conclusions justified based on the evidence, were the figures or tables useful and easy to understand)?  
    - Fig.1 is quite difficult to interpret due to the vast amount of information provided and requires quite a bit of time to analyze    
    - Fig.3 is a bit confusing to understand due to the small markers and black/white scheme  


### Module_3 essay

Assessing environmental diversity is a complex process. Not all organisms grow under the same growth conditions. Thus, many microbial species are â€œunculturableâ€ in the laboratory and are difficult to detect without sequence analysis. Metagenomic analysis is a method used to define microbial species based on sequence or functional similarity. The analysis of an environmental sample may be done by the analysis of genome or of conserved genetic regions such as the 16S ribosomal RNA (rRNA) sequence. However, many complications are involved in this process, particularly due to horizontal gene transfer (HGT) that occurs between species. In this essay, we will discuss the challenges that occur when defining species caused by HGT, how HGT maintains global biogeochemical cycles and why it is important to have a clear definition of microbial species.  

Defining microbial species is a challenging process due to microbial evolution, particularly as a result of HGT. In metagenomics, mixed reads from different species in an environment are binned into operational taxonomic units (OTUs) based on DNA sequence similarity to a set threshold (1). OTUs correspond to microbial species, which can allow researchers to determine species abundance in an environment. Specific genes can also be binned by their functional characteristics based on functional gene anchors, in order to interpret sets of genes (1). However, the challenge involved in this process is that it is unclear from which species the binned gene originated. Thus, it creates difficulty tracing the origin of a certain gene or metabolic pathway. Distinguishing species becomes more complicated when genes are transferred between microbial species by means of HGT and recombination. Currently, species are broadly defined, in that there is a difference between strains with regards to physiology, genome content and ecology (2). Periodic selection that occurs can cause extinction for an ecotype, while also allowing another ecotype that has obtained a favourable gene by HGT to replace the original strain or species (2). Due to the frequent HGT events, species may switch form a niche to another and acquire metabolic genes required for survival. With the ability to acquire and donate genes, it is challenging to define a microbial species because the characteristics of the species, such as ecology and expressed genes, can frequently change.  

HGT influences maintenance of global biogeochemical cycles through time by allowing the dispersal of essential genes required for survival to be transferred to other organisms in the environment. Due to HGT, organisms are able to transfer genes to other related species (3). For example, in the nitrogen (N) cycle, the gene for nitrogenise has been transferred multiple times across taxa throughout history (3, 4). Specifically, the nif gene that is involved in nitrogen fixation has been suggested to be acquired through HGT in the cyanobacterium, Microcoleus chthonoplastes (5). Another example is the sulfur oxygenase reductase (SOR) gene used in the sulfur cycle found in the Sulfolobales taxa (6). Homologs of the gene were found in Acidianus tengchongensis and other microbes, suggesting that the SOR gene was acquired via HGT (6). Although HGT is only common among closely related species and for simple pathways requiring few enzymes, the distribution of these genes can prevent the extinction of these important genes. As previously mentioned, HGT can cause niche differentiation (2). This may help maintain the global biogeochemical cycles by not outcompeting species that can perform an important metabolic reaction of the cycle. Although HGT complicates the process of defining a species, it assists in the maintenance of biogeochemical cycles.

It is necessary to have a clear definition of species because it provides researchers and scientists and agreed upon species name for important purposes such as scientific analysis or clinical diagnosis. With a defined name for a species, important characteristics and information about the species would be universal for researchers and scientists around the world (7). This would allow clinical microbiologists and physicians to integrate findings from research on microbial disease to make consistent clinical diagnosis (7). For example, the species name Salmonella typhi, would give a microbiologist immediate knowledge of the disease they produce, the shape of the bacteria and their natural environment. In addition, scientists would be able to make meaningful comparisons on the microbial diversity of different samples and allow research groups analyzing environmental samples to have less difficulty collaborating due to consistent species naming. For example, in the â€œCandyâ€ in-class activity in MICB 425, each group named their candy sample differently. This resulted in confusion and some difficulties compiling data to make a comparison between the candy samples due to the inconsistent naming system among multiple independent groups. A standard definition of species would resolve the issue and would also allow for a more organized microbial database. Thus, having a clear definition of a microbial species is necessary to avoid confusion and make research more efficient.  

Microbial evolution due to HGT makes analyzing environmental microbial diversity and defining microbial species a complex task for microbiologists. HGT allows the transfer of genes or operons to related species. This complicates the process of defining a microbial species due to the changing characteristics of species caused by gene acquisition and the selective environment. However, HGT can allow the maintenance of global biogeochemical cycles by propagating important genes of the cycles to other species and by niche differentiation to reduce competition between species. Although HGT renders defining species challenging, it is still essential to have a clear definition of species due to communication purposes between and within the scientific and medical community.  


### Module 03 references
Utilize this space to include a bibliography of any literature you want associated with this module. We recommend keeping this as the final header under each module.

  - Torres-Beltrán, M., Hawley, A. K., Capelle, D., Zaikova, E., Walsh, D. A., Mueller, A., ... & Finke, J. (2017). A compendium of geochemical information from the Saanich Inlet water column. Scientific data, 4, 170159. [link](https://www.nature.com/articles/sdata2017159)  
  - Sogin, M. L., Morrison, H. G., Huber, J. A., Welch, D. M., Huse, S. M., Neal, P. R., ... & Herndl, G. J. (2006). Microbial diversity in the deep sea and the underexplored "rare biosphere". Proceedings of the National Academy of Sciences, 103(32), 12115-12120. [link](http://www.pnas.org/content/103/32/12115.short)  
  - Welch, R. A., Burland, V., Plunkett, G. I. I. I., Redford, P., Roesch, P., Rasko, D., ... & Stroud, D. (2002). Extensive mosaic structure revealed by the complete genome sequence of uropathogenic Escherichia coli. Proceedings of the National Academy of Sciences, 99(26), 17020-17024. [link](http://www.pnas.org/content/99/26/17020.short)  
  - Hawley, A. K., Torres-Beltrán, M., Zaikova, E., Walsh, D. A., Mueller, A., Scofield, M., ... & Shevchuk, O. (2017). A compendium of multi-omic sequence information from the Saanich Inlet water column. Scientific data, 4, 170160. [link](https://www.nature.com/articles/sdata2017160)  
  - Kunin, V., Engelbrektson, A., Ochman, H., & Hugenholtz, P. (2010). Wrinkles in the rare biosphere: pyrosequencing errors can lead to artificial inflation of diversity estimates. Environmental microbiology, 12(1), 118-123. [link](https://onlinelibrary.wiley.com/doi/full/10.1111/j.1462-2920.2009.02051.x)  
  - Hallam, S. J., Torres-Beltrán, M., & Hawley, A. K. (2017). Monitoring microbial responses to ocean deoxygenation in a model oxygen minimum zone. Scientific data, 4. [link](https://www.nature.com/articles/sdata2017158)  
  - Callahan, B. J., McMurdie, P. J., & Holmes, S. P. (2017). Exact sequence variants should replace operational taxonomic units in marker-gene data analysis. The ISME journal, 11(12), 2639. [link](https://www.nature.com/articles/ismej2017119)  
  - Thompson, J. R., Pacocha, S., Pharino, C., Klepac-Ceraj, V., Hunt, D. E., Benoit, J., ... & Polz, M. F. (2005). Genotypic diversity within a natural coastal bacterioplankton population. Science, 307(5713), 1311-1313. [link](http://science.sciencemag.org/content/307/5713/1311)  

References for essay 3:  
1. Eisen, J. 2007. Environmental shotgun sequencing: Its potential and challenges for studying the hidden world of microbes. Plos Biology. 5:384-388. doi: 10.1371/journal.pbio.0050082.  
2. Wiedenbeck, J, Cohan, FM. 2011. Origins of bacterial diversity through horizontal genetic transfer and adaptation to new ecological niches. FEMS Microbiol. Rev. 35:957-976. doi: 10.1111/j.1574-6976.2011.00292.x.  
3. Schimel, J, Schaeffer, S. 2012. Microbial control over carbon cycling in soil. Frontiers in Microbiology. 3:348. doi: 10.3389/fmicb.2012.00348.  
4. Falkowski, PG, Fenchel, T, Delong, EF. 2008. The Microbial Engines That Drive Earthâ€™s Biogeochemical Cycles. Science. 320:1034-1039. doi: 10.1126/science.1153213.  
5. Bolhuis, H, Severin, I, Confurius-Guns, V, Wollenzien, UIA, Stal, LJ. 2010; 2009. Horizontal transfer of the nitrogen fixation gene cluster in the cyanobacterium Microcoleus chthonoplastes. ISME Journal. 4:121-130. doi: 10.1038/ismej.2009.99.  
6. Blank, CE. 2012. Low rates of lateral gene transfer among metabolic genes define the evolving biogeochemical niches of Archaea through deep time. Archaea. 2012:23. doi: http://dx.doi.org/10.1155/2012/843539.  
7. Baron, EJ. Classification. In: Baron S, editor. Medical Microbiology. 4th edition. Galveston (TX): University of Texas Medical Branch at Galveston; 1996. Chapter 3. Available from: https://www.ncbi.nlm.nih.gov/books/NBK8406/  

---

# Module 04  


## Distributed Metabolic Networks

---


### Project_2  

####Abstract  

Genes encoding key steps in the nitrogen cycle are well defined and provide a basis for functional anchor screening to determine their distribution across prokaryotic taxa. In this study, Saanich Inlet was used as a model ecosystem to investigate the metabolic coupling and symbiotic interactions that influence the nitrogen cycle in oxygen minimum zones. Water samples were collected at seven major depths spanning the oxycline, and genomic DNA and RNA were extracted and sequenced. The resulting reads were processed, assembled, and analyzed using the Tree-based Sensitive and Accurate Protein Profiler (TreeSAPP) pipeline to reconstruct the nitrogen cycle along defined redox gradients in Saanich Inlet. The *narG* gene was then investigated in detail. Its DNA levels were found to increase with depth while RNA levels decreased with depth. Proteobacteria contributed most *narG* DNA, while Actinobacteria contributed most RNA. Further, *narG* RNA abundance increased along with nitrite concentrations in the Inlet, but had the opposite relationship with nitrate concentrations; this was the expected result given that *narG* mediates the conversion of nitrate to nitrite. Overall, *narG* is responsible for a conversion taking place the beginning of the nitrogen cycle, and this provides reasoning for its presence at the given depths, while providing a key piece to exemplify evolutionary and environmental reasoning for the distributed metabolism seen in the nitrogen cycle.

####Introduction  

Oxygen minimum zones (OMZs) are regions in the ocean where dissolved oxygen concentrations fall below 20 $\mu$M (1). Because of temperature increases and other effects caused by global warming, OMZs are rising at a notable rate. Saanich Inlet is a seasonally anoxic fjord off the coast of British Columbia and a model ecosystem for studying the diversity and biochemical responses of microbial communities to the hypoxic environments commonly observed in OMZs (1, 2). In particular, Saanich Inlet has been used to model the metabolic coupling and symbiotic interactions that occur in OMZs (3).  The inlet undergoes recurring cycles of water column stratification and deep water renewal, making it a model ecosystem for studying microbial responses to changes in oceanic oxygenation levels (4). Increased levels of primary productivity in ocean surfaces during the spring season, along with the limited mixing which occurs between the basin and surface waters both result in the development of an anoxic body of water with increasing depth in the Inlet (2). These anoxic regions become highly populated with chemolithoautotrophs, eventually leading to a decrease in aerobically respiring organisms found deeper within these zones. Past studies have shown that these types of metabolic shifts usually lead to a loss of nitrogen along with the production of notable greenhouse gases such as methane (CH~4~) and nitrous oxide (N~2~O) (1). Some species have also been shown to engage in sulfide (H~2~S) oxidation and nitrate (NO~3~^-^) reduction pathways within these zones (5). 

The nitrogen cycle--the biogeochemical cycle by which nitrogenous compounds are interconverted between chemical forms for environmental circulation--consists mainly of nitrogen fixation, nitrification and denitrification, and has been catalyzed by microorganisms long before the appearance of humans (6). Even today, much of it is still dictated by the diverse microbial niches present in the surrounding environment. An example of this control can be seen in denitrification--the conversion of NO~3~^-^ to nitrogen gas (N~2~)--carried out by denitrifying bacteria. Denitrication is a highly important process as it prevents the accumulation of nitrogen compounds to toxic levels that could lead to eutrophication, and maintains the homeostasis of nitrogen distribution between soil and atmosphere. Having an atmospheric residence time of approximately 1 billion years, nitrogen gas (N~2~) is highly inert and is not accessible for the synthesis of proteins and nucleic acids, though this problem is easily solved by the conversion of N~2~ to NH~4~^+^ through nitrogen fixation (6).  

The aim of our project was to characterize the involvement of our gene of interest, *narG*, which is known for its role in the reduction of NO~3~^-^ to NO~2~^-^ in the denitrification cycle. Recent literature has highlighted that microbial taxa do not necessarily integrate whole elemental cycles into their individual metabolic pathways, but often divide these reactions among the community so that consortia of taxa must rely on each other to fulfill their metabolic requirements (2).  
  

####Methods  
Water samples were collected from Saanich Inlet at depths 10, 100, 120, 135, 150, 165, and 200m spanning the oxycline during Saanich Inlet Cruise 72. The samples were subsequently filtered through a 0.22 $\mu$m Sterivex filter to collect biomass and both genomic DNA and RNA were extracted from these samples. The extracted RNA was converted into cDNA. Both total genomic DNA and cDNA were used to construct the shotgun Illumina libraries. Sequencing data were generated on the Illumina HiSeq platform with 2x150bp technology. Further sampling and sequencing details can be found in *A compendium of multi-omic sequence information from the Saanich Inlet water column* by Hawkley et al. (2017).  

The IMG/M pipeline was then used to process the resulting reads. Metapathways 2.5 was used to assemble and process the metagenomes. A beta version of Tree-based Sensitive and Accurate Protein Profiler (TreeSAPP) pipeline was used to reconstruct the nitrogen cycle along defined redox gradients in Saanich Inlet using Google Cloud services. iTOL version 4.0 was used to generate phylogenetic trees based on the processed data. The contig maps were loaded into R version 3.4.3. Tidyverse and Cowplot packages were loaded and used to complete the data analysis.  


####Results  

#####Environment setup and Data Cleaning  
```{r echo=FALSE}
library(tidyverse)
library(cowplot)
library(phyloseq)
```

Loading, parsing (initial data contains data for all genes - we are only interested in *narG*), renaming data.  We only need taxonomy, abundance, and query data.
```{r message=FALSE, echo=FALSE}
narG.DNA.10m = read_tsv("narG/marker_contig/DNA10m_marker_contig_map.tsv") %>%
    filter(Marker=="narG") %>%
    select(Tax.DNA.10 = Confident_Taxonomy, Abund.DNA.10 = Abundance, Query)

narG.RNA.10m = read_tsv("narG/marker_contig/RNA10m_marker_contig_map.tsv") %>%
    filter(Marker=="narG") %>% 
    select(Tax.RNA.10 = Confident_Taxonomy, Abund.RNA.10 = Abundance, Query)

narG.DNA.100m = read_tsv("narG/marker_contig/DNA100m_marker_contig_map.tsv") %>%
    filter(Marker=="narG") %>% 
    select(Tax.DNA.100 = Confident_Taxonomy, Abund.DNA.100 = Abundance, Query)

narG.RNA.100m = read_tsv("narG/marker_contig/RNA100m_marker_contig_map.tsv") %>%
    filter(Marker=="narG") %>%  
    select(Tax.RNA.100 = Confident_Taxonomy, Abund.RNA.100 = Abundance, Query)

narG.DNA.120m = read_tsv("narG/marker_contig/DNA120m_marker_contig_map.tsv") %>%
    filter(Marker=="narG") %>%  
    select(Tax.DNA.120 = Confident_Taxonomy, Abund.DNA.120 = Abundance, Query)

narG.RNA.120m = read_tsv("narG/marker_contig/RNA120m_marker_contig_map.tsv") %>%
    filter(Marker=="narG") %>%  
    select(Tax.RNA.120 = Confident_Taxonomy, Abund.RNA.120 = Abundance, Query)

narG.DNA.135m = read_tsv("narG/marker_contig/DNA135m_marker_contig_map.tsv") %>%
    filter(Marker=="narG") %>%  
    select(Tax.DNA.135 = Confident_Taxonomy, Abund.DNA.135 = Abundance, Query)

narG.RNA.135m = read_tsv("narG/marker_contig/RNA135m_marker_contig_map.tsv") %>%
    filter(Marker=="narG") %>%  
    select(Tax.RNA.135 = Confident_Taxonomy, Abund.RNA.135 = Abundance, Query)

narG.DNA.150m = read_tsv("narG/marker_contig/DNA150m_marker_contig_map.tsv") %>%
    filter(Marker=="narG") %>%  
    select(Tax.DNA.150 = Confident_Taxonomy, Abund.DNA.150 = Abundance, Query)

narG.RNA.150m = read_tsv("narG/marker_contig/RNA150m_marker_contig_map.tsv") %>% 
    filter(Marker=="narG") %>% 
    select(Tax.RNA.150 = Confident_Taxonomy, Abund.RNA.150 = Abundance, Query)

narG.DNA.165m = read_tsv("narG/marker_contig/DNA165m_marker_contig_map.tsv") %>% 
    filter(Marker=="narG") %>% 
    select(Tax.DNA.165 = Confident_Taxonomy, Abund.DNA.165 = Abundance, Query)

narG.RNA.165m = read_tsv("narG/marker_contig/RNA165m_marker_contig_map.tsv") %>% 
    filter(Marker=="narG") %>% 
    select(Tax.RNA.165 = Confident_Taxonomy, Abund.RNA.165 = Abundance, Query)

narG.DNA.200m = read_tsv("narG/marker_contig/DNA200m_marker_contig_map.tsv") %>% 
    filter(Marker=="narG") %>% 
    select(Tax.DNA.200 = Confident_Taxonomy, Abund.DNA.200 = Abundance, Query)

narG.RNA.200m = read_tsv("narG/marker_contig/RNA200m_marker_contig_map.tsv") %>% 
    filter(Marker=="narG") %>% 
    select(Tax.RNA.200 = Confident_Taxonomy, Abund.RNA.200 = Abundance, Query)
```

Data manipulation into a single data frame
```{r, echo=FALSE}
narG.all = narG.DNA.10m %>% 
# Combine the data frames will full_join to keep all the data
  full_join(narG.RNA.10m, by = "Query") %>%
  full_join(narG.DNA.100m, by = "Query") %>% 
  full_join(narG.RNA.100m, by = "Query") %>%
  full_join(narG.DNA.120m, by = "Query") %>% 
  full_join(narG.RNA.120m, by = "Query") %>%
  full_join(narG.DNA.135m, by = "Query") %>% 
  full_join(narG.RNA.135m, by = "Query") %>%
  full_join(narG.DNA.150m, by = "Query") %>% 
  full_join(narG.RNA.150m, by = "Query") %>%
  full_join(narG.DNA.165m, by = "Query") %>% 
  full_join(narG.RNA.165m, by = "Query") %>% 
  full_join(narG.DNA.200m, by = "Query") %>% 
  full_join(narG.RNA.200m, by = "Query") %>% 
# Create a taxonomy variable aggregating all taxonomy columns so as to fill in any NAs that might occur. !is.na means "is not NA", so the following says that the Taxonomy data should be taken from Tax.RNA.10 if that is not NA, or else take it from Tax.DNA.10 if that is not NA, or else Tax.RNA.200, etc. until if all are NA, give Taxonomy of "unclassified"
  mutate(Taxonomy = ifelse(!is.na(Tax.RNA.10), Tax.RNA.10,
                    ifelse(!is.na(Tax.DNA.10), Tax.DNA.10,
                    ifelse(!is.na(Tax.RNA.100), Tax.RNA.100,
                    ifelse(!is.na(Tax.DNA.100), Tax.DNA.100,
                    ifelse(!is.na(Tax.RNA.120), Tax.RNA.120,
                    ifelse(!is.na(Tax.DNA.120), Tax.DNA.120,
                    ifelse(!is.na(Tax.RNA.135), Tax.RNA.135,
                    ifelse(!is.na(Tax.DNA.135), Tax.DNA.135,
                    ifelse(!is.na(Tax.RNA.150), Tax.RNA.150,
                    ifelse(!is.na(Tax.DNA.150), Tax.DNA.150,
                    ifelse(!is.na(Tax.RNA.165), Tax.RNA.165,
                    ifelse(!is.na(Tax.DNA.165), Tax.DNA.165,
                    ifelse(!is.na(Tax.RNA.200), Tax.RNA.200,
                    ifelse(!is.na(Tax.DNA.200), Tax.DNA.200,
                           "unclassified"))))))))))))))) %>% 
# Get rid of the old Tax variables
  select(-starts_with("Tax.")) %>% 
# Gather all the abundance data into 1 column 
  gather("Key", "Abundance", starts_with("Abund")) %>% 
# Turn the Key into Depth and RNA/DNA variables. We can easily do this because we specifically named these variables with period separation when we loaded in the original data
  separate(Key, c("Key","Type","Depth_m"), by = ".") %>% 
# Remove Key variable now that it only contains "Abund". This also serves to reorder the columns so that the very long Query is at the end.
  select(Depth_m, Type, Abundance, Taxonomy, Query) %>% 
# Make sure R knows depth is numerical since it came from a character variable
  mutate(Depth_m = as.numeric(Depth_m)) %>% 
# Separate Taxonomy into columns so we can plot at different levels
  separate(Taxonomy, into = c("Domain", "Phylum", "Class", "Order", "Family", "Genus", "Species"), sep="; ")
```

Warning above ignored, because not all queries could be classified down to species level, so these cells are filled as NA, which is fine.

##### Final data table
Table 1. Final data of *narG* gene 
```{r, echo=FALSE}
narG.all
```

##### 1. How does the DNA abundance of *narG* gene differ with depth?  

Figure 1 indicates that the DNA abundance of the *narG* gene has an overall increasing trend with depth. The DNA abundance of the *narG* gene increases gradually from 21.40 to 990.72 at depth from 10 m to 150 m. However, it declines to 249.21 at depth of 165 m, but increases again to 717.90 at depth of 200 m.  

```{r, echo=FALSE}
narG.all %>% 
# Filter the DNA data
  filter(Type == "DNA") %>% 
# Change NAs to "unclassified" at the level you want to plot. Here we will do Genus
  mutate(Genus = ifelse(is.na(Phylum), "unclassified", Phylum)) %>% 
  group_by(Depth_m) %>%
  mutate(abund_sum=sum(Abundance, na.rm=TRUE))%>%
  group_by(Depth_m)%>%
  summarize(abund_sum=mean(abund_sum))%>%

  
ggplot(aes(x = "narG", y = Depth_m)) +
# Use the size aesthetic to show abundance
  geom_point(aes(size = abund_sum)) +
# Reverse the why axis so depth increases going down
  scale_y_reverse(lim=c(200,0)) +
  labs(y="Depth (m)")+
  theme_classic() +
  scale_size_continuous(name="Abundance",breaks = c(200,400,600,800)) +
  geom_text(aes(label=abund_sum, hjust=1.5))+
  labs(caption = "Figure 1. Abundance of the narG gene (DNA) at different depths",
       x = "") +
  theme(plot.caption = element_text(hjust=0.5, size=rel(1.0)))
```
 

##### 2. What taxa are responsible for *narG*? Are they the same with depth? With DNA versus RNA?  

RNA abundance of the *narG* gene with depth does not match the trend of DNA abundance. Figure 2 shows that RNA abundance of the *narG* gene increases significantly from 10 m to 100 m, and reaches its second highest level of 3585.45 at 100 m. Interestingly, the RNA abundance fluctuates between a range of 2500-3700 at depths 100-135 m. After that, it decreases gradually until 200 m. On the other hand, RNA abundance of *narG* gene is much larger than the DNA abundance at depth of 100-165 m, while less *narG* RNA is found in the samples from depth of 10 m and 200 m.  
```{r, echo=FALSE}
# DNA and RNA abundance with depth
DNA.abund.depth=narG.all %>% 
  filter(Type == "DNA") %>% 
  mutate(Genus = ifelse(is.na(Phylum), "unclassified", Phylum)) %>% 
  group_by(Depth_m) %>%
  mutate(abund_sum=sum(Abundance, na.rm=TRUE))%>%
  group_by(Depth_m)%>%
  summarize(DNA=mean(abund_sum))

RNA.abund.depth=narG.all %>% 
  filter(Type == "RNA") %>% 
  mutate(Genus = ifelse(is.na(Phylum), "unclassified", Phylum)) %>% 
  group_by(Depth_m) %>%
  mutate(abund_sum=sum(Abundance, na.rm=TRUE))%>%
  group_by(Depth_m)%>%
  summarize(RNA=mean(abund_sum))

left_join(DNA.abund.depth,RNA.abund.depth,by="Depth_m")%>%
  gather(Type,abund_sum,DNA:RNA)%>%

# Show both RNA and DNA using an x variable  
ggplot(aes(x = Type, y = Depth_m)) +
  geom_point(aes(size = abund_sum)) +
  labs(y="Depth (m)")+
  scale_y_reverse(lim=c(200,0)) +
  theme_classic()+
  scale_size_continuous(name="Abundance",breaks = c(500,1000,1500,2000,3000)) +
  geom_text(mapping=aes(label=abund_sum),hjust=1.2)+
  labs(caption = "Figure 2. Abundance of the narG gene (DNA vs. RNA) at different depths") +
  theme(plot.caption = element_text(hjust=0.5, size=rel(1.0)))
```

##### 3. What taxa are responsible for *narG* gene? Are they the same with depth? With DNA versus RNA?  

As shown in Figure 3, Actinobacteria, candidatus Omnitrophica, Chlorobi, Euryarchaeota, metagenomes, Proteobacteria, and unclassified Bacteria are responsible for *narG* DNA. Among of them, Actinobacteria, candidatus Omnitrophica, Chlorobi, Euryarchaeota, and metagenomes have evenly distributed DNA abundance across depth above 100 m. However, the distribution of Proteobacterial DNA abundance with depth follows a similar trend with that of *narG* DNA abundance, in which abundance increases from depth of 10 m to 150 m, declines at depth of 150-165 m, and finally increases again at 200 m. On the contrary, DNA abundance of unclassified Bacteria is the same at depth below 165 m but increases slightly at 200 m.  

Actinobacteria, Proteobacteria, and unclassified Bacteria make dominant contributions to *narG* RNA among the 11 phyla. Actinobacteria is the predominant phylum for *narG* RNA contribution at depth of 100 m and 120 m, but its RNA abundance decreases gradually at deeper water and eventually becomes undetectable at depth of 200 m. Both Proteobacteria and unclassified Bacteria have observed RNA at the seven different depth, although their RNA abundances are not distributed in the same trend with depth. Proteobacterial RNA abundances at depth of 150 m and 200 m are higher than that from other depths, while unclassified Bacteria have the lowest RNA abundance at 150 m and 200 m. Additionally, unclassified Bacteria have RNA abundance increasing with depth at shallow water and reaching the highest point at depth of 135 m.


```{r fig.width=8, echo=FALSE}
narG.all %>% 
# Change NAs to "unclassified" at the level you want to plot
  mutate(Phylum = ifelse(is.na(Phylum), "unclassified Bacteria", Phylum)) %>% 
  
ggplot(aes(x = Phylum, y = Depth_m)) +
# Use an ifelse statement to make 0 values into NAs so that they don't show up on the plot
# Use position_dodge to keep points from overlapping
  geom_point(aes(size = ifelse(Abundance == 0, NA, Abundance), color = Type), position = position_dodge(0.5)) +
  scale_y_reverse(lim=c(200,0)) +
  labs(y="Depth (m)")+
  theme_classic() +
# Rename legend
  scale_size_continuous(name = "Abundance") +  
  theme(axis.text.x = element_text(angle = 45, hjust =1, size=rel(1.2)))+
  labs(caption = "Figure 3. Abundance of phyla with narG (DNA vs. RNA) at different depths") +
  theme(plot.caption = element_text(hjust=0.5, size=rel(1.1)))
 
```


##### 4. How does the abundance of *narG* gene relate to nitrogen species in Saanich?  

The *narG* gene has significant impacts on nitrite (NO~2~^-^) and nitrate (NO~3~^-^) concentration across depth in Saanich. It is responsible for the reduction of nitrate to nitrite. As a result, when the RNA abundance of *narG* gene remains at a high level at depths 100-135m (Figure 2), nitrate concentration stops increasing and declines dramatically from its highest point at depth of 100 m (Figure 4D). On the other hand, due to poor nitrification in deeper water, although RNA of *narG* gene is less abundant at depth above 150 m, nitrate concentration continues decreasing with depth until to 0 at depth of 165 m. Similarly, *narG* is also one of the key factors for the fluctuation of nitrite concentration at depth 100-165 m. The low ammonium concentration in depth of 10-135 m indicates a high level of nitrification in the shallow water (Figure 4A), which is responsible for the significant increase of nitrate and the decrease of nitrite at depth 10-100 m. However, nitrite concentration does not keep decreasing with depth above 100m, instead, it remains around 0.09 $\mu$M as a result of high nitrate reduction level caused by the abundant RNA of *narG* gene. Therefore, *narG* gene abundance is closely related to nitrite and nitrate concentration in Saanich.  

```{r fig.width=7, fig.height=7, echo=FALSE}
load("mothur_phyloseq.RData")

metadata = data.frame(mothur@sam_data)

# DNA/RNA abundance with depth
plot1 = narG.all %>% 
# Change NAs to "unclassified" at the level you want to plot
  mutate(Phylum = ifelse(is.na(Phylum), "unclassified", Phylum)) %>% 
  
ggplot(aes(x = Phylum, y = Depth_m)) +
  geom_point(aes(size = ifelse(Abundance == 0, NA, Abundance), color = Type), position = position_dodge(0.5)) +
  scale_y_reverse(lim=c(200,0)) +
  labs(y = "Depth (m)") +
  theme_classic() +
  scale_size_continuous(name = "Abundance")+ 
  theme(axis.text.x = element_text(angle = 45, hjust =1))

#Then, we make our plots for NO2, "plot2", and NO3, "plot3"

plot2 = metadata %>% 
# Order the data by depth  
  arrange(Depth_m) %>% 
  
ggplot(aes(x = NO2_uM, y = Depth_m)) +
    geom_point() +
    geom_path(aes(group = 1)) +
    scale_y_reverse(lim=c(200,0)) +
  theme_classic() +
  labs(y = "Depth (m)",
       x = "NO2 (uM)")
plot3 = metadata %>% 
# Order the data by depth  
  arrange(Depth_m) %>% 
  
ggplot(aes(x = NO3_uM, y = Depth_m)) +
    geom_point() +
    geom_path(aes(group = 1)) +
    scale_y_reverse(lim=c(200,0)) +
  theme_classic() +
  labs(y = "Depth (m)",
       x = "NO3 (uM)")

plot4 = metadata %>% 
# Order the data by depth  
  arrange(Depth_m) %>% 
  
ggplot(aes(x = NH4_uM, y = Depth_m)) +
    geom_point() +
    geom_path(aes(group = 1)) +
    scale_y_reverse(lim=c(200,0)) +
  theme_classic() +
  labs(y = "Depth (m)",
       x = "NH4 (uM)")

plot5 = metadata %>% 
# Order the data by depth  
  arrange(Depth_m) %>% 
  
ggplot(aes(x = N2O_nM, y = Depth_m)) +
    geom_point() +
    geom_path(aes(group = 1)) +
    scale_y_reverse(lim=c(200,0)) +
  theme_classic() +
  labs(y = "Depth (m)",
       x = "N2O (nM)")

plot_grid(plot4, plot5, plot2, plot3, labels=c("A", "B", "C", "D"), axis="l", nrow=2, ncol=2, rel_widths=c(1/2, 1/2, 1/2,1/2))+
  labs(caption = "Figure 4. Nitrogen species in Saanich across depth. 
A) Ammonium (NH4) concentration across depth   B) Nitrous oxide (N2O) concentration across depth 
C) Nitrite (NO2) concentration across depth     D) Nitrate (NO3) concentration across depth") +
  theme(plot.caption = element_text(hjust=0.5, size=rel(0.75)))
```

####Discussion  

In anthropogenic times, NO~3~^-^ levels have significantly increased in environmental water systems resulting from agricultural activities (7). With increases in NO~3~^-^ entering these systems, the accumulation of nitrogen compounds above threshold levels can lead to eutrophication. Microbe-driven processes such as denitrification, are able to counteract this phenomenon by converting NO~3~^-^ to N~2~. In this context, our gene of interest, *narG*, specifically mediates the first step of this pathway. As the alpha subunit and catalytic domain of the membrane-bound dissimilatory nitrate reductase, *narG* is responsible for the conversion of NO~3~^-^ to NO~2~^-^. NO~2~^-^ will become the substrate of *nirK* and *nirS*, and through subsequent steps and enzymes, eventually lead to the production of N~2~ (8). Interestingly, it has been found that the process of transporting nitrate into the cytoplasmic location of the active site of NarG is inhibited by oxygen, making denitrification an anaerobic process (9). As a result, it is expected that there is greater amounts of *narG* transcription in OMZs compared to surface level waters. From figure 2, peak transcription of *narG* is found at 100 m where oxygen levels are approaching OMZ conditions (Figure 5). Additionally, the large abundance of *narG* transcription at 100 m and deeper can be explained with the corresponding abundance of NO~3~^-^ substrate. Many denitrifying facultative anaerobic microorganisms respire NO~3~^-^ in the absence of oxygen. Denitrification is considered to be the highest energy-yielding respiration system in anoxic environments (10). Increases in NO~3~^-^ at anoxic depth leads to the reduction of nitrate to nitrite in the cytoplasm by *narG*. It has been found that this process is coupled to the translocation of protons into the periplasm which directly contributes to a proton-motive force for energy conservation (8). Therefore, it is not surprising that *narG* transcription levels are near absent at oxic depths where oxygen can still be used as the preferential terminal electron acceptor (TEA), and abundant at anoxic depths where NO~3~^-^ can be used as an alternative TEA.  

```{r fig.width=5, fig.height=5, echo=FALSE}
# Oxygen with depth
metadata %>% 
# Order the data by depth  
  arrange(Depth_m) %>% 
  
ggplot(aes(x = O2_uM, y = Depth_m)) +
    geom_point() +
    geom_path(aes(group = 1)) +
    scale_y_reverse(lim=c(200,0)) +
  theme_classic() +
  labs(y = "Depth (m)",
       x = "O2 (uM)")+
  labs(caption="Figure 5. Oxygen in Saanich across depth")+
  theme(plot.caption = element_text(hjust=0.5,size=rel(1.0)))
```

Metabolisms in the nitrogen cycle is widely distributed, such that each reaction step is catalyzed by a specific enzyme from a different bacterial species carrying the corresponding genes. This distribution may be the consequence of the distinct energy and nutrient sources which are available in different ecosystems, as well as other mechanisms, namely horizontal gene transfer, and adaptive gene loss.  

In aerobic environments like shallow water and topsoil, nitrification is the predominant process of the nitrogen cycle, due to the availability of ammonia and nitrite as energy sources for nitrifying bacteria and the inhibitive effect of oxygen on denitrifiers (11,12). On the contrary, denitrification occurs in anoxic environments including OMZs and marine sediments, where nitrate serves as energy and nutrient source and the limitation of oxygen on denitrification is minimum (8). Besides, some bacteria evolve additional mechanisms to protect nitrogen-related enzymes from unfavorable environments. For example, *Trichodesmium* spp. and *Nodularia* spp. adopt the strategy that nitrogen fixation is separated from photosynthesis, in order to minimize oxygen suppression on nitrogenases, an enzyme for nitrogen fixation (8). Therefore, environmental conditions is one of the possible reason for the distributed metabolisms.  

Another potential explanation for metabolic distribution of the nitrogen cycle is horizontal gene transfer (HGT). Bacteria may acquire nitrogen-related genes from other species through HGT and take up the niche of corresponding reactions in the nitrogen cycle. For instance, ammonia monooxygenase genes, encoding the key enzyme required for ammonia oxidation, are observed widely distributed among different bacterial species (6). As a result, if a certain bacterial species responsible for ammonia oxidation became extinct in a specific ecosystem, other species, carrying ammonia monooxygenase genes and able to perform the process of ammonia oxidation, would take up the niche of the extinct one.  On the other hand, the retention of horizontally transferred gene is largely driven by selective pressure on nutrients and energy, and thus different processes in the nitrogen cycle are widely distributed among microorganisms and ecosystems (6).   

Gene loss may also make contributions to the wide distribution of metabolisms in the nitrogen cycle. A hypothesis, called as Black Queen Hypothesis (BQH), is proposed recently to explain the dependency of free-living microorganisms having metabolic gene loss on other species (13). In this hypothesis, two groups of bacterial species are defined: "helper" and "beneficiaries". The hypothesis predicts that in a microbial community, "beneficiaries", which is selected to lose a costly and dispensable functions, will obtain the function from a fraction of other individuals in the community, until the function is sufficient to support the whole community (13). Based on the hypothesis, it can be explained that why nitrogen fixation is performed by a small fraction of microorganisms in the ocean. The possible reasons may be that genes responsible for nitrogen fixation were selectively lost during evolution and fixed N was turned into a public good (13).  

This study paves the way for future investigations into metabolic coupling and symbiotic interactions in the nitrogen cycle. It appears that *narG* levels are regulated by oxygen levels and availability of the NO~3~^-^ substrate; however, the effect of other environmental factors has yet to be related with gene abundance. According to Palmer et al., in sediment communities, Actinobacterial  *narG* dominates in extreme environments characterized by low pH or high temperature, while Proteobacterial *narG* is present in pH-neutral soils (15). Given that Actinobacteria and Proteobacteria dominated DNA and transcript expression of *narG* in an aquatic context, it would be useful to determine whether these abundances are due to pH, temperature, or other measurable geochemical parameters.  

Additionally, as *narG* is inhibited by its own product, nitrite, it may be interesting to see how *narG* abundances vary with *nirK* and *nirS* abundances, which consume nitrite. Interestingly, previous studies have noted that *nirS* was dominated by $\beta$-Proteobacteria, while *nirK* was dominated by $\alpha$-Proteobacteria (14). Our study results also suggest *narG* is dominated in part by Proteobacteria, although more in depth analysis into different classes of Proteobacteria have not been attempted. A deeper analysis into the abundance of *narG* in different classes of Proteobacteria may be necessary to establish a correlation between how the abundances of *nirK*, *nirS*, and *narG* vary within the Proteobacteria phylum.  


####References  

1. Walsh DA, Zaikova E, Howes CG, Song YC, Wright JJ, Tringe SG, Hallam SJ. 2009. Metagenome of a versatile chemolithoautotroph from expanding oceanic dead zones. Science 326(5952): 578-582.  
2. Torres-Beltrán M, Hawley AK, Capelle D, Zaikova E, Walsh DA., Mueller A, Finke J. 2017. A compendium of geochemical information from the Saanich Inlet water column. Nature scientific data 4(170159).  
3. Hawley AK, Torres-Beltrán M, Zaikova E, Walsh DA, Mueller A, Scofield M, Kheirandish S, Payne C, Pakhomova L, Bhatia M, Shevchek O, Gies EA, Fairley D, Malfatii SA, Norbeck AD, Brewer HM, Pasa-Tolic, L, del Rio TG, Suttle CA, Trige S, Hallam SJ. Data Descriptor: A compendium of multi-omic sequence information from the Saanich Inlet water column. Nature scientific data 4(170160).  
4. Hallam SJ, Torres-Beltrán M, Hawley AK. Comment: Monitoring microbial responses to ocean deoxygenation in a model oxygen minimum zone. Nature scientific data 4(170158).  
5. National Research Council. 2007. The new science of metagenomics: revealing the secrets of our microbial planet. National Academies Press, Washington, DC.  
6. Falkowski PG, Fenchel T, Delong EF. 2008. The microbial engines that drive Earth's biogeochemical cycles. Science 320(5879): 1034-1039.  
7. Smith CJ, Nedwell DB, Dong LF, Osborn AM. 2007. Diversity and Abundance of Nitrate Reductase Genes (narG and napA), Nitrite Reductase Genes (nirS and nrfA), and Their Transcripts in Estuarine Sediments. Applied and environmental microbiology, 73(11): 3612-3622.  
8. Kuypers MM., Marchant, HK,  Kartal B. (2018). The microbial nitrogen-cycling network. Nature Reviews Microbiology.  
9. Moreno-Vivián C, Cabello P, Martínez-Luque M, Blasco R, Castillo F. (1999). Prokaryotic nitrate reduction: molecular properties and functional distinction among bacterial nitrate reductases. Journal of bacteriology, 181(21): 6573-6584.   
10. Strohm TO, Griffin B, Zumft WG, Schink B. (2007). Growth yields in bacterial denitrification and nitrate ammonification. Applied and environmental microbiology, 73(5): 1420-1424.  
11. Ward BB. 2008. Nitrification,  p 2511-2518. In Jørgensen SE and Fath BD(ed), Ecological Processes, vol. 3 of Encyclopedia of Ecology, 5 vols. Oxford: Elsevier.  
12. Nakajima M, Hayamize T, Nishimura H. 1983. Effect of oxygen concentration on the rates of denitrification and denitrification in the sediments of an eutrophic lake. Water Resources, vol. 18, no. 3, p 335-337.  
13. Morris JJ, Lenski RE, Zinser ER. 2012. The black queen hypothesis: Evolution of dependencies through adaptive gene loss. mBio 3(2). DOI: 10.1128/mBio.00036-12.
14. Yu Z, Yang J, Liu L. 2014. Denitrifier Community in the Oxygen Minimum Zone of a Subtropical Deep Reservoir. PLoS One 9(3): e92055.  
15. Palmer K, Horn MA. 2015. Denitrification Activity of a Remarkably Diverse Fen Denitrifier Community in Finnish Lapland Is N-Oxide Limited. PLoS One 10(4): e0123123.  
