---
title: "MICB425 Portfolio - Microbial Ecological Genomics: From Genomes to Biomes, Spring 2018"
author: "Albert Chang (26234147)"
date: "version `r format(Sys.time(), '%B %d, %Y')`"
output: 
  html_document:
    toc: true
---
# Module 03
Reserve the first level headings (#) for the start of a new Module. This will help to organize your portfolio in an intuitive fashion.  
**Note: Please edit this template to your heart's content. This is meant to be the armature upon which you build your individual portfolio. You do not need to keep this instructive text in your final portfolio, although you do need to keep module and assignment names so we can identify what is what.**


## Module 03 portfolio check
The first of your second level headers (##) is to be used for the portfolio content checks. The Module 01 portfolio check has been built for you directly into this template, but will also be available as a stand-alone markdown document available on the MICB425 GitHub so that you know what is required in each module section in your portfolio. The completion status and comments will be filled in by the instructors during portfolio checks when your current portfolios are pulled from GitHub.

* Evidence worksheet_01
    + Completion status:
    + Comments:
* Evidence worksheet_02
    + Completion status:
    + Comments:
* Evidence worksheet_03
    + Completion status:
    + Comments:
* Problem Set_01
    + Completion status:
    + Comments:
* Problem Set_02
    + Completion status:
    + Comments:
* Writing assessment_02
    + Completion status:
    + Comments:
* Additional Readings
    + Completion status:
    + Comments  
    


### Portfolio repo setup
Detail the code you used to create, initialize, and push your portfolio repo to GitHub. This will be helpful as you will need to repeat many of these steps to update your porfolio throughout the course.


## Module 3 title

---
### Problem set_03 "Metagenomics: Genomic Analysis of Microbial Communities"
#### Learning objectives: 
Specific emphasis should be placed on the process used to find the answer. Be as comprehensive as possible e.g. provide URLs for web sources, literature citations, etc.  
*(Reminders for how to format links, etc in RMarkdown are in the RMarkdown Cheat Sheets)*

#### Specific Questions:
* How many prokaryotic divisions have been described and how many have no cultured representatives (microbial dark matter)?  
    - Solden et al. (2016). "The Bright Side of Microbial Dark Matter: Lessons Learned from the Uncultivated Majority". -> 89 Bacterial and 20 Archaeal phyla recogized by small rRNA subunits
        - only 0.1 - 1% of microbes in environment have been cultured
    - Youssef et al. (2015). "Assessing the Global Phylum Level Diversity Within the Bacterial Domain: A Review"
        - 70 bacterial phyla with no cultured representatives (this number may be greater)
        - previously confined to pure cultures, found 12 bacterial phyla with cultured representatives: Actinobacteria, Bacteroidetes, Chlamydiae, Chlorobi, Chloroflexi, Cyanobacteria, Thermi (Deinococcus-Thermus), Firmicutes, Planctomycetes, Proteobacteria, Spirochaetes, Thermotogae
  

* How many metagenome sequencing projects are currently available in the public domain and what types of environments are they sourced from?  
    - From EBI Metagenomics (Europe)
        - 1486 public projects
        - 86201 samples
        - soil, marine, grassland, fecal, rumens, agriculture, etc.
    - JGI (U.S.)
        - 36715 public projects

* What types of on-line resources are available for warehousing and/or analyzing environmental sequence information (provide names, URLS and applications)?  
    - MEGAN (metagenome analyzer)
        - http://ab.inf.uni-tuebingen.de/software/megan5/download/welcome.html
        - analyze metagenomes and group using taxonomic analysis
    - IMG/M (Integrated Microbial Genomes)
        - http://img.jgi.doe.gov/cgi-bin/m/main.cgi
        - tool for analyzing publicly available genome/metagenome datasets
    - MG-RAST (metagenomics analysis server)
        - https://www.mg-rast.org/
    - Tag-based sequencing: SSU rRNA -> MOTHUR, QIIME
    - BLAST (NCBI)

* What is the difference between phylogenetic and functional gene anchors and how can they be used in metagenome analysis?  
    - Phylogenetic anchors:
        - link other unknown genes within genome to a taxon
        - collection of genes can be placed into bins of discrete taxonomic groups
        - onl one copy per cell; ensures no HGT, and genes only appear once in tree
        - Looks at size of bins for relative abundance of taxa
    - Functional anchors:
        - tells what the cell does
        - search for gene that codes for protin involved in the terminal steps of a pathway
        - can count abundance of phylogenetic anchor

* What is metagenomic sequence binning? What types of algorithmic approaches are used to produce sequence bins? What are some risks and opportunities associated with using sequence bins for metabolic reconstruction of uncultivated microorganisms?  
    - binning: grouping reads or contigs and assigning them to operational taxonomic units (OTU)
        - Taxonomy dependent - classifies DNA fragments by comparing to reference database
        - Taxonomy independent - reference free binning by clustering similar features (e.g. GC content, etc.)
    - sequence composition based binning -> genomic signatures
        - Interpolated Markov Model (IMM)
    - abundance based binning
        - Expectation-Maximization (EM) algorithm
    -Hybrid binning
        - Lander-Waterman model
    Sedlar et al. (2016) "Bioinformatics strategies for taxonomy independent binning and visualization of sequence in shotgun metagenomics"

* Is there an alternative to metagenomic shotgun sequencing that can be used to access the metabolic potential of uncultivated microorganisms? What are some risks and opportunities associated with this alternative?  
    - Douterelo et al. (2014). "Methodological approaches for studying the microbial ecology of drinking water distribution systems."
    - Clone dependent shotgun library
        - Find functions of proteins from the gene of uncultivable microbes by cloning gene fragments into an expression host
        - Limitation: gene may not be compatible with the host
    - Pacific Bioscience
    - Nanopore
    - Metatranscriptomes
        - RT-qPCR:
            -high sensitivity, can run multiple samples + analyze different genes at once
            - quantitative representation of changes in gene expression due to different treaments/controls
            - Limitations:
                - RNA is highly unstable
                - DNA extracted needs to be pure
        - Functional Microarrays:
            - Overall assessment of gene expression within a microbial community
            - simultaneous evaluation of many mRNA (rapid)
            - Limitations:
                - liquid chromatography/2D gels for protein separation can be tedious
                - Must use LC-MS (Liquid chromatography-mass spectrometry) to characterize samples



### Module 02 references
Utilize this space to include a bibliography of any literature you want associated with this module. We recommend keeping this as the final header under each module.

An example for Whitman and Wiebe (1998) has been included below.

Whitman WB, Coleman DC, and Wiebe WJ. 1998. Prokaryotes: The unseen majority. Proc Natl Acad Sci USA. 95(12):6578â€“6583.  [PMC33863](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC33863/)
